From ce6ad4396d8d8b44a16c0f4e17316ee37d6665cd Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Wed, 20 Aug 2014 00:38:39 +0200
Subject: [PATCH] update hwdb

(cherry picked from commit cccc74971fe3c853832d742a15eac94f365603b6)

Conflicts:
	hwdb/20-OUI.hwdb
	hwdb/20-pci-vendor-model.hwdb
	hwdb/20-usb-vendor-model.hwdb
---
 hwdb/20-OUI.hwdb              | 555 +++++++++++++++++++++++++++++++++++--
 hwdb/20-pci-vendor-model.hwdb | 625 +++++++++++++++++++++++++++++++++++++++++-
 hwdb/20-usb-vendor-model.hwdb | 220 ++++++++++++++-
 3 files changed, 1360 insertions(+), 40 deletions(-)

diff --git a/hwdb/20-OUI.hwdb b/hwdb/20-OUI.hwdb
index 97f1e6b2c0..e0c5fed985 100644
--- a/hwdb/20-OUI.hwdb
+++ b/hwdb/20-OUI.hwdb
@@ -21371,7 +21371,7 @@ OUI:000A2F*
  ID_OUI_FROM_DATABASE=Artnix Inc.
 
 OUI:000A30*
- ID_OUI_FROM_DATABASE=Johnson Controls-ASG
+ ID_OUI_FROM_DATABASE=Visteon Corporation
 
 OUI:000A31*
  ID_OUI_FROM_DATABASE=HCV Consulting
@@ -23288,7 +23288,7 @@ OUI:000CB1*
  ID_OUI_FROM_DATABASE=Salland Engineering (Europe) BV
 
 OUI:000CB2*
- ID_OUI_FROM_DATABASE=Comstar Co., Ltd.
+ ID_OUI_FROM_DATABASE=UNION co., ltd.
 
 OUI:000CB3*
  ID_OUI_FROM_DATABASE=ROUND Co.,Ltd.
@@ -23528,7 +23528,7 @@ OUI:000D01*
  ID_OUI_FROM_DATABASE=P&E Microcomputer Systems, Inc.
 
 OUI:000D02*
- ID_OUI_FROM_DATABASE=NEC AccessTechnica, Ltd.
+ ID_OUI_FROM_DATABASE=NEC Platforms, Ltd.
 
 OUI:000D03*
  ID_OUI_FROM_DATABASE=Matrics, Inc.
@@ -30311,7 +30311,7 @@ OUI:0015DF*
  ID_OUI_FROM_DATABASE=Clivet S.p.A.
 
 OUI:0015E0*
- ID_OUI_FROM_DATABASE=ST-Ericsson
+ ID_OUI_FROM_DATABASE=Ericsson
 
 OUI:0015E1*
  ID_OUI_FROM_DATABASE=Picochip Ltd
@@ -32375,7 +32375,7 @@ OUI:001893*
  ID_OUI_FROM_DATABASE=SHENZHEN PHOTON BROADBAND TECHNOLOGY CO.,LTD
 
 OUI:001894*
- ID_OUI_FROM_DATABASE=zimocom
+ ID_OUI_FROM_DATABASE=NPCore, Inc.
 
 OUI:001895*
  ID_OUI_FROM_DATABASE=Hansun Technologies Inc.
@@ -33887,7 +33887,7 @@ OUI:001A8B*
  ID_OUI_FROM_DATABASE=CHUNIL ELECTRIC IND., CO.
 
 OUI:001A8C*
- ID_OUI_FROM_DATABASE=Astaro AG
+ ID_OUI_FROM_DATABASE=Sophos Ltd
 
 OUI:001A8D*
  ID_OUI_FROM_DATABASE=AVECS Bergen GmbH
@@ -34652,7 +34652,7 @@ OUI:001B8A*
  ID_OUI_FROM_DATABASE=2M Electronic A/S
 
 OUI:001B8B*
- ID_OUI_FROM_DATABASE=NEC AccessTechnica, Ltd.
+ ID_OUI_FROM_DATABASE=NEC Platforms, Ltd.
 
 OUI:001B8C*
  ID_OUI_FROM_DATABASE=JMicron Technology Corp.
@@ -35066,7 +35066,7 @@ OUI:001C14*
  ID_OUI_FROM_DATABASE=VMware, Inc
 
 OUI:001C15*
- ID_OUI_FROM_DATABASE=TXP Corporation
+ ID_OUI_FROM_DATABASE=iPhotonix LLC
 
 OUI:001C16*
  ID_OUI_FROM_DATABASE=ThyssenKrupp Elevator
@@ -35975,7 +35975,7 @@ OUI:001D43*
  ID_OUI_FROM_DATABASE=Shenzhen G-link Digital Technology Co., Ltd.
 
 OUI:001D44*
- ID_OUI_FROM_DATABASE=KROHNE
+ ID_OUI_FROM_DATABASE=KROHNE Messtechnik GmbH
 
 OUI:001D45*
  ID_OUI_FROM_DATABASE=CISCO SYSTEMS, INC.
@@ -36863,7 +36863,7 @@ OUI:001E6B*
  ID_OUI_FROM_DATABASE=Cisco SPVTG
 
 OUI:001E6C*
- ID_OUI_FROM_DATABASE=Carbon Mountain LLC
+ ID_OUI_FROM_DATABASE=Opaque Systems
 
 OUI:001E6D*
  ID_OUI_FROM_DATABASE=IT R&D Center
@@ -38471,7 +38471,7 @@ OUI:002084*
  ID_OUI_FROM_DATABASE=OCE PRINTING SYSTEMS, GMBH
 
 OUI:002085*
- ID_OUI_FROM_DATABASE=EXIDE ELECTRONICS
+ ID_OUI_FROM_DATABASE=Eaton Corporation
 
 OUI:002086*
  ID_OUI_FROM_DATABASE=MICROTECH ELECTRONICS LIMITED
@@ -44312,7 +44312,7 @@ OUI:003A9C*
  ID_OUI_FROM_DATABASE=CISCO SYSTEMS, INC.
 
 OUI:003A9D*
- ID_OUI_FROM_DATABASE=NEC AccessTechnica, Ltd.
+ ID_OUI_FROM_DATABASE=NEC Platforms, Ltd.
 
 OUI:003AAF*
  ID_OUI_FROM_DATABASE=BlueBit Ltd.
@@ -46394,7 +46394,7 @@ OUI:0060B8*
  ID_OUI_FROM_DATABASE=CORELIS Inc.
 
 OUI:0060B9*
- ID_OUI_FROM_DATABASE=NEC Infrontia Corporation
+ ID_OUI_FROM_DATABASE=NEC Platforms, Ltd
 
 OUI:0060BA*
  ID_OUI_FROM_DATABASE=SAHARA NETWORKS, INC.
@@ -49015,6 +49015,9 @@ OUI:00A1DE*
 OUI:00A2DA*
  ID_OUI_FROM_DATABASE=INAT GmbH
 
+OUI:00A2F5*
+ ID_OUI_FROM_DATABASE=Guangzhou Yuanyun Network Technology Co.,Ltd
+
 OUI:00A2FF*
  ID_OUI_FROM_DATABASE=abatec group AG
 
@@ -49036,6 +49039,9 @@ OUI:00AA70*
 OUI:00ACE0*
  ID_OUI_FROM_DATABASE=ARRIS Group, Inc.
 
+OUI:00AEFA*
+ ID_OUI_FROM_DATABASE=Murata Manufacturing Co., Ltd.
+
 OUI:00B009*
  ID_OUI_FROM_DATABASE=Grass Valley Group
 
@@ -51595,6 +51601,9 @@ OUI:00E666*
 OUI:00E6D3*
  ID_OUI_FROM_DATABASE=NIXDORF COMPUTER CORP.
 
+OUI:00E6E8*
+ ID_OUI_FROM_DATABASE=Netzin Technology Corporation,.Ltd.
+
 OUI:00E8AB*
  ID_OUI_FROM_DATABASE=Meggitt Training Systems, Inc.
 
@@ -51610,9 +51619,15 @@ OUI:00F051*
 OUI:00F403*
  ID_OUI_FROM_DATABASE=Orbis Systems Oy
 
+OUI:00F46F*
+ ID_OUI_FROM_DATABASE=Samsung Elec Co.,Ltd
+
 OUI:00F4B9*
  ID_OUI_FROM_DATABASE=Apple
 
+OUI:00F76F*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:00F860*
  ID_OUI_FROM_DATABASE=PT. Panggung Electric Citrabuana
 
@@ -51844,6 +51859,12 @@ OUI:0494A1*
 OUI:0498F3*
  ID_OUI_FROM_DATABASE=ALPS Electric Co,. Ltd.
 
+OUI:0499E6*
+ ID_OUI_FROM_DATABASE=Shenzhen Yoostar Technology Co., Ltd
+
+OUI:049B9C*
+ ID_OUI_FROM_DATABASE=Eadingcore  Intelligent Technology Co., Ltd.
+
 OUI:049C62*
  ID_OUI_FROM_DATABASE=BMT Medical Technology s.r.o.
 
@@ -52423,6 +52444,9 @@ OUI:0808C2*
 OUI:0808EA*
  ID_OUI_FROM_DATABASE=AMSC
 
+OUI:0809B6*
+ ID_OUI_FROM_DATABASE=Masimo Corp
+
 OUI:080C0B*
  ID_OUI_FROM_DATABASE=SysMik GmbH Dresden
 
@@ -52438,6 +52462,9 @@ OUI:080EA8*
 OUI:080FFA*
  ID_OUI_FROM_DATABASE=KSP INC.
 
+OUI:08115E*
+ ID_OUI_FROM_DATABASE=Bitel Co., Ltd.
+
 OUI:081196*
  ID_OUI_FROM_DATABASE=Intel Corporate
 
@@ -52597,6 +52624,9 @@ OUI:087D21*
 OUI:088039*
  ID_OUI_FROM_DATABASE=Cisco SPVTG
 
+OUI:0881BC*
+ ID_OUI_FROM_DATABASE=HongKong Ipro Technology Co., Limited
+
 OUI:0881F4*
  ID_OUI_FROM_DATABASE=Juniper Networks
 
@@ -52693,9 +52723,15 @@ OUI:08E5DA*
 OUI:08E672*
  ID_OUI_FROM_DATABASE=JEBSEE ELECTRONICS CO.,LTD.
 
+OUI:08E84F*
+ ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
+
 OUI:08EA44*
  ID_OUI_FROM_DATABASE=Aerohive Networks, Inc.
 
+OUI:08EB29*
+ ID_OUI_FROM_DATABASE=Jiangsu Huitong Group Co.,Ltd.
+
 OUI:08EB74*
  ID_OUI_FROM_DATABASE=Humax
 
@@ -52705,6 +52741,9 @@ OUI:08EBED*
 OUI:08EDB9*
  ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
 
+OUI:08EE8B*
+ ID_OUI_FROM_DATABASE=Samsung Elec Co.,Ltd
+
 OUI:08EF3B*
  ID_OUI_FROM_DATABASE=MCS Logic Inc.
 
@@ -52759,6 +52798,9 @@ OUI:0C17F1*
 OUI:0C191F*
  ID_OUI_FROM_DATABASE=Inform Electronik
 
+OUI:0C1DAF*
+ ID_OUI_FROM_DATABASE=Beijing Xiaomi communications co.,ltd
+
 OUI:0C1DC2*
  ID_OUI_FROM_DATABASE=SeAH Networks
 
@@ -52942,6 +52984,9 @@ OUI:0CAC05*
 OUI:0CAF5A*
  ID_OUI_FROM_DATABASE=GENUS POWER INFRASTRUCTURES LIMITED
 
+OUI:0CB319*
+ ID_OUI_FROM_DATABASE=Samsung Elec Co.,Ltd
+
 OUI:0CB4EF*
  ID_OUI_FROM_DATABASE=Digience Co.,Ltd.
 
@@ -53009,7 +53054,7 @@ OUI:0CD996*
  ID_OUI_FROM_DATABASE=CISCO SYSTEMS, INC.
 
 OUI:0CD9C1*
- ID_OUI_FROM_DATABASE=Johnson Controls-ASG
+ ID_OUI_FROM_DATABASE=Visteon Corporation
 
 OUI:0CDA41*
  ID_OUI_FROM_DATABASE=Hangzhou H3C Technologies Co., Limited
@@ -53080,6 +53125,9 @@ OUI:1001CA*
 OUI:1005CA*
  ID_OUI_FROM_DATABASE=Cisco
 
+OUI:1008B1*
+ ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
+
 OUI:10090C*
  ID_OUI_FROM_DATABASE=Janome Sewing Machine Co., Ltd.
 
@@ -53200,6 +53248,9 @@ OUI:104B46*
 OUI:104D77*
  ID_OUI_FROM_DATABASE=Innovative Computer Engineering
 
+OUI:104E07*
+ ID_OUI_FROM_DATABASE=Shanghai Genvision Industries Co.,Ltd
+
 OUI:105172*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
@@ -53234,7 +53285,7 @@ OUI:1065CF*
  ID_OUI_FROM_DATABASE=IQSIM
 
 OUI:106682*
- ID_OUI_FROM_DATABASE=NEC AccessTechnica, Ltd.
+ ID_OUI_FROM_DATABASE=NEC Platforms, Ltd.
 
 OUI:10683F*
  ID_OUI_FROM_DATABASE=LG Electronics
@@ -53401,6 +53452,9 @@ OUI:10F96F*
 OUI:10F9EE*
  ID_OUI_FROM_DATABASE=Nokia Corporation
 
+OUI:10FACE*
+ ID_OUI_FROM_DATABASE=Reacheng Communication Technology Co.,Ltd
+
 OUI:10FBF0*
  ID_OUI_FROM_DATABASE=KangSheng LTD.
 
@@ -53668,6 +53722,9 @@ OUI:14F42A*
 OUI:14F65A*
  ID_OUI_FROM_DATABASE=Xiaomi inc.
 
+OUI:14F893*
+ ID_OUI_FROM_DATABASE=Wuhan FiberHome Digital Technology Co.,Ltd.
+
 OUI:14FEAF*
  ID_OUI_FROM_DATABASE=SAGITTAR LIMITED
 
@@ -54190,6 +54247,9 @@ OUI:1C7C45*
 OUI:1C7CC7*
  ID_OUI_FROM_DATABASE=Coriant GmbH
 
+OUI:1C7E51*
+ ID_OUI_FROM_DATABASE=3bumen.com
+
 OUI:1C7EE5*
  ID_OUI_FROM_DATABASE=D-Link International
 
@@ -54220,6 +54280,9 @@ OUI:1C955D*
 OUI:1C959F*
  ID_OUI_FROM_DATABASE=Veethree Electronics And Marine LLC
 
+OUI:1C965A*
+ ID_OUI_FROM_DATABASE=Weifang goertek Electronics CO.,LTD
+
 OUI:1C973D*
  ID_OUI_FROM_DATABASE=PRICOM Design
 
@@ -54229,6 +54292,9 @@ OUI:1C994C*
 OUI:1C9C26*
  ID_OUI_FROM_DATABASE=Zoovel Technologies
 
+OUI:1C9ECB*
+ ID_OUI_FROM_DATABASE=Beijing Nari Smartchip Microelectronics Company Limited
+
 OUI:1CA2B1*
  ID_OUI_FROM_DATABASE=ruwido austria gmbh
 
@@ -54254,7 +54320,7 @@ OUI:1CB094*
  ID_OUI_FROM_DATABASE=HTC Corporation
 
 OUI:1CB17F*
- ID_OUI_FROM_DATABASE=NEC AccessTechnica, Ltd.
+ ID_OUI_FROM_DATABASE=NEC Platforms, Ltd.
 
 OUI:1CB243*
  ID_OUI_FROM_DATABASE=TDC A/S
@@ -54478,6 +54544,9 @@ OUI:2074CF*
 OUI:207600*
  ID_OUI_FROM_DATABASE=Actiontec Electronics, Inc
 
+OUI:207693*
+ ID_OUI_FROM_DATABASE=Lenovo (Beijing) Limited.
+
 OUI:207C8F*
  ID_OUI_FROM_DATABASE=Quanta Microsystems,Inc.
 
@@ -54766,6 +54835,9 @@ OUI:249442*
 OUI:249504*
  ID_OUI_FROM_DATABASE=SFR
 
+OUI:2497ED*
+ ID_OUI_FROM_DATABASE=Techvision Intelligent Technology Limited
+
 OUI:24A2E1*
  ID_OUI_FROM_DATABASE=Apple, Inc
 
@@ -55306,6 +55378,9 @@ OUI:2C27D7*
 OUI:2C282D*
  ID_OUI_FROM_DATABASE=BBK COMMUNICATIAO TECHNOLOGY CO.,LTD.
 
+OUI:2C2997*
+ ID_OUI_FROM_DATABASE=Microsoft Corporation
+
 OUI:2C2D48*
  ID_OUI_FROM_DATABASE=bct electronic GesmbH
 
@@ -55327,6 +55402,9 @@ OUI:2C36F8*
 OUI:2C3731*
  ID_OUI_FROM_DATABASE=ShenZhen Yifang Digital Technology Co.,LTD
 
+OUI:2C3796*
+ ID_OUI_FROM_DATABASE=CYBO CO.,LTD.
+
 OUI:2C3996*
  ID_OUI_FROM_DATABASE=SAGEMCOM
 
@@ -55360,6 +55438,9 @@ OUI:2C441B*
 OUI:2C44FD*
  ID_OUI_FROM_DATABASE=Hewlett Packard
 
+OUI:2C5089*
+ ID_OUI_FROM_DATABASE=Shenzhen Kaixuan Visual Technology Co.,Limited
+
 OUI:2C534A*
  ID_OUI_FROM_DATABASE=Shenzhen Winyao Electronic Limited
 
@@ -55390,6 +55471,9 @@ OUI:2C5D93*
 OUI:2C5FF3*
  ID_OUI_FROM_DATABASE=Pertronic Industries
 
+OUI:2C600C*
+ ID_OUI_FROM_DATABASE=QUANTA COMPUTER INC.
+
 OUI:2C625A*
  ID_OUI_FROM_DATABASE=Finest Security Systems Co., Ltd
 
@@ -55549,12 +55633,18 @@ OUI:2CEDEB*
 OUI:2CEE26*
  ID_OUI_FROM_DATABASE=Petroleum Geo-Services
 
+OUI:2CF0EE*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:2CF203*
  ID_OUI_FROM_DATABASE=EMKO ELEKTRONIK SAN VE TIC AS
 
 OUI:2CF4C5*
  ID_OUI_FROM_DATABASE=Avaya, Inc
 
+OUI:2CF7F1*
+ ID_OUI_FROM_DATABASE=Seeed Technology Inc.
+
 OUI:2CFAA2*
  ID_OUI_FROM_DATABASE=Alcatel-Lucent
 
@@ -55570,6 +55660,9 @@ OUI:300D2A*
 OUI:300ED5*
  ID_OUI_FROM_DATABASE=Hon Hai Precision Ind.Co.Ltd
 
+OUI:3010B3*
+ ID_OUI_FROM_DATABASE=Liteon Technology Corporation
+
 OUI:3010E4*
  ID_OUI_FROM_DATABASE=Apple, Inc.
 
@@ -55705,12 +55798,18 @@ OUI:306E5C*
 OUI:3071B2*
  ID_OUI_FROM_DATABASE=Hangzhou Prevail Optoelectronic Equipment Co.,LTD.
 
+OUI:307350*
+ ID_OUI_FROM_DATABASE=Inpeco SA
+
 OUI:307512*
  ID_OUI_FROM_DATABASE=Sony Mobile Communications AB
 
 OUI:30766F*
  ID_OUI_FROM_DATABASE=LG Electronics
 
+OUI:3077CB*
+ ID_OUI_FROM_DATABASE=Maike Industry(Shenzhen)CO.,LTD
+
 OUI:30786B*
  ID_OUI_FROM_DATABASE=TIANJIN Golden Pentagon Electronics Co., Ltd.
 
@@ -55831,6 +55930,9 @@ OUI:30F7D7*
 OUI:30F9ED*
  ID_OUI_FROM_DATABASE=Sony Corporation
 
+OUI:30FAB7*
+ ID_OUI_FROM_DATABASE=Tunai Creative
+
 OUI:30FD11*
  ID_OUI_FROM_DATABASE=MACROTECH (USA) INC.
 
@@ -55897,6 +55999,9 @@ OUI:343111*
 OUI:3431C4*
  ID_OUI_FROM_DATABASE=AVM GmbH
 
+OUI:3438AF*
+ ID_OUI_FROM_DATABASE=Inlab Software GmbH
+
 OUI:3440B5*
  ID_OUI_FROM_DATABASE=IBM
 
@@ -55909,6 +56014,9 @@ OUI:344B3D*
 OUI:344B50*
  ID_OUI_FROM_DATABASE=ZTE Corporation
 
+OUI:344DEA*
+ ID_OUI_FROM_DATABASE=zte corporation
+
 OUI:344DF7*
  ID_OUI_FROM_DATABASE=LG Electronics
 
@@ -55942,6 +56050,9 @@ OUI:346178*
 OUI:346288*
  ID_OUI_FROM_DATABASE=Cisco
 
+OUI:3464A9*
+ ID_OUI_FROM_DATABASE=Hewlett Packard
+
 OUI:34684A*
  ID_OUI_FROM_DATABASE=Teraworks Co., Ltd.
 
@@ -56152,6 +56263,9 @@ OUI:34E2FD*
 OUI:34E42A*
  ID_OUI_FROM_DATABASE=Automatic Bar Controls Inc.
 
+OUI:34E6AD*
+ ID_OUI_FROM_DATABASE=Intel Corporate
+
 OUI:34EF44*
  ID_OUI_FROM_DATABASE=2Wire
 
@@ -56200,6 +56314,9 @@ OUI:380B40*
 OUI:380DD4*
  ID_OUI_FROM_DATABASE=Primax Electronics LTD.
 
+OUI:380E7B*
+ ID_OUI_FROM_DATABASE=V.P.S. Thai Co., Ltd
+
 OUI:380F4A*
  ID_OUI_FROM_DATABASE=Apple
 
@@ -56236,12 +56353,18 @@ OUI:3826CD*
 OUI:3828EA*
  ID_OUI_FROM_DATABASE=Fujian Netcom Technology Co., LTD
 
+OUI:382C4A*
+ ID_OUI_FROM_DATABASE=ASUSTek COMPUTER INC.
+
 OUI:382DD1*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
 OUI:3831AC*
  ID_OUI_FROM_DATABASE=WEG
 
+OUI:383BC8*
+ ID_OUI_FROM_DATABASE=2wire
+
 OUI:383F10*
  ID_OUI_FROM_DATABASE=DBL Technology Ltd.
 
@@ -56329,6 +56452,9 @@ OUI:388EE7*
 OUI:3891FB*
  ID_OUI_FROM_DATABASE=Xenox Holding BV
 
+OUI:389496*
+ ID_OUI_FROM_DATABASE=Samsung Elec Co.,Ltd
+
 OUI:389592*
  ID_OUI_FROM_DATABASE=Beijing Tendyron Corporation
 
@@ -56518,6 +56644,9 @@ OUI:3C1A79*
 OUI:3C1CBE*
  ID_OUI_FROM_DATABASE=JADAK LLC
 
+OUI:3C1E13*
+ ID_OUI_FROM_DATABASE=HANGZHOU SUNRISE TECHNOLOGY CO., LTD
+
 OUI:3C25D7*
  ID_OUI_FROM_DATABASE=Nokia Corporation
 
@@ -56989,6 +57118,9 @@ OUI:40984E*
 OUI:40987B*
  ID_OUI_FROM_DATABASE=Aisino Corporation
 
+OUI:409B0D*
+ ID_OUI_FROM_DATABASE=Shenzhen Yourf Kwan Industrial Co., Ltd
+
 OUI:409FC7*
  ID_OUI_FROM_DATABASE=BAEKCHUN I&C Co., Ltd.
 
@@ -57253,6 +57385,9 @@ OUI:446C24*
 OUI:446D57*
  ID_OUI_FROM_DATABASE=Liteon Technology Corporation
 
+OUI:446D6C*
+ ID_OUI_FROM_DATABASE=Samsung Elec Co.,Ltd
+
 OUI:44700B*
  ID_OUI_FROM_DATABASE=IFFU
 
@@ -57367,6 +57502,9 @@ OUI:44C56F*
 OUI:44C9A2*
  ID_OUI_FROM_DATABASE=Greenwald Industries
 
+OUI:44CE7D*
+ ID_OUI_FROM_DATABASE=SFR
+
 OUI:44D15E*
  ID_OUI_FROM_DATABASE=Shanghai Kingto Information Technology Ltd
 
@@ -57685,6 +57823,9 @@ OUI:4C09B4*
 OUI:4C0B3A*
  ID_OUI_FROM_DATABASE=TCT Mobile Limited
 
+OUI:4C0BBE*
+ ID_OUI_FROM_DATABASE=Microsoft
+
 OUI:4C0DEE*
  ID_OUI_FROM_DATABASE=JABIL CIRCUIT (SHANGHAI) LTD.
 
@@ -57703,6 +57844,9 @@ OUI:4C1480*
 OUI:4C14A3*
  ID_OUI_FROM_DATABASE=TCL Technoly Electronics (Huizhou) Co., Ltd.
 
+OUI:4C16F1*
+ ID_OUI_FROM_DATABASE=zte corporation
+
 OUI:4C17EB*
  ID_OUI_FROM_DATABASE=SAGEMCOM
 
@@ -57730,6 +57874,9 @@ OUI:4C26E7*
 OUI:4C2C80*
  ID_OUI_FROM_DATABASE=Beijing Skyway Technologies Co.,Ltd
 
+OUI:4C2C83*
+ ID_OUI_FROM_DATABASE=Zhejiang KaNong Network Technology Co.,Ltd.
+
 OUI:4C2F9D*
  ID_OUI_FROM_DATABASE=ICM Controls
 
@@ -57754,6 +57901,9 @@ OUI:4C3B74*
 OUI:4C3C16*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
+OUI:4C48DA*
+ ID_OUI_FROM_DATABASE=Beijing Autelan Technology Co.,Ltd
+
 OUI:4C4B68*
  ID_OUI_FROM_DATABASE=Mobile Device, Inc.
 
@@ -57943,6 +58093,9 @@ OUI:4CDF3D*
 OUI:4CE1BB*
  ID_OUI_FROM_DATABASE=Zhuhai HiFocus Technology Co., Ltd.
 
+OUI:4CE2F1*
+ ID_OUI_FROM_DATABASE=sclak srl
+
 OUI:4CE676*
  ID_OUI_FROM_DATABASE=Buffalo Inc.
 
@@ -58267,6 +58420,9 @@ OUI:50ED94*
 OUI:50F003*
  ID_OUI_FROM_DATABASE=Open Stack, Inc.
 
+OUI:50F43C*
+ ID_OUI_FROM_DATABASE=Leeo Inc
+
 OUI:50F520*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
@@ -58300,6 +58456,9 @@ OUI:540536*
 OUI:54055F*
  ID_OUI_FROM_DATABASE=Alcatel Lucent
 
+OUI:54098D*
+ ID_OUI_FROM_DATABASE=deister electronic GmbH
+
 OUI:54112F*
  ID_OUI_FROM_DATABASE=Sulzer Pump Solutions Finland Oy
 
@@ -58459,6 +58618,9 @@ OUI:549F35*
 OUI:54A04F*
  ID_OUI_FROM_DATABASE=t-mac Technologies Ltd
 
+OUI:54A050*
+ ID_OUI_FROM_DATABASE=ASUSTek COMPUTER INC.
+
 OUI:54A31B*
  ID_OUI_FROM_DATABASE=Shenzhen Linkworld Technology Co,.LTD
 
@@ -58525,6 +58687,9 @@ OUI:54E3B0*
 OUI:54E43A*
  ID_OUI_FROM_DATABASE=Apple, Inc.
 
+OUI:54E4BD*
+ ID_OUI_FROM_DATABASE=FN-LINK TECHNOLOGY LIMITED
+
 OUI:54E63F*
  ID_OUI_FROM_DATABASE=ShenZhen LingKeWeiEr Technology Co., Ltd.
 
@@ -58603,6 +58768,9 @@ OUI:581FAA*
 OUI:581FEF*
  ID_OUI_FROM_DATABASE=Tuttnaer LTD
 
+OUI:582136*
+ ID_OUI_FROM_DATABASE=KMB systems, s.r.o.
+
 OUI:58238C*
  ID_OUI_FROM_DATABASE=Technicolor CH USA
 
@@ -58705,6 +58873,9 @@ OUI:587BE9*
 OUI:587E61*
  ID_OUI_FROM_DATABASE=Hisense Electric Co., Ltd
 
+OUI:587FB7*
+ ID_OUI_FROM_DATABASE=SONAR INDUSTRIAL CO., LTD.
+
 OUI:587FC8*
  ID_OUI_FROM_DATABASE=S2M
 
@@ -58903,6 +59074,9 @@ OUI:5C260A*
 OUI:5C2AEF*
  ID_OUI_FROM_DATABASE=Open Access Pty Ltd
 
+OUI:5C2BF5*
+ ID_OUI_FROM_DATABASE=Vivint
+
 OUI:5C2E59*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
@@ -59017,9 +59191,15 @@ OUI:5C93A2*
 OUI:5C95AE*
  ID_OUI_FROM_DATABASE=Apple
 
+OUI:5C966A*
+ ID_OUI_FROM_DATABASE=RTNET
+
 OUI:5C969D*
  ID_OUI_FROM_DATABASE=Apple
 
+OUI:5C97F3*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:5C9AD8*
  ID_OUI_FROM_DATABASE=Fujitsu Limited
 
@@ -59032,6 +59212,9 @@ OUI:5CA3EB*
 OUI:5CA48A*
  ID_OUI_FROM_DATABASE=Cisco
 
+OUI:5CAAFD*
+ ID_OUI_FROM_DATABASE=Sonos, Inc.
+
 OUI:5CAC4C*
  ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
 
@@ -59167,6 +59350,9 @@ OUI:600308*
 OUI:600347*
  ID_OUI_FROM_DATABASE=Billion Electric Co. Ltd.
 
+OUI:600417*
+ ID_OUI_FROM_DATABASE=POSBANK CO.,LTD
+
 OUI:600F77*
  ID_OUI_FROM_DATABASE=SilverPlus, Inc
 
@@ -59407,6 +59593,9 @@ OUI:60C547*
 OUI:60C5A8*
  ID_OUI_FROM_DATABASE=Beijing LT Honway Technology Co.,Ltd
 
+OUI:60C798*
+ ID_OUI_FROM_DATABASE=Verifone, Inc.
+
 OUI:60C980*
  ID_OUI_FROM_DATABASE=Trymus
 
@@ -59773,6 +59962,9 @@ OUI:64AE0C*
 OUI:64AE88*
  ID_OUI_FROM_DATABASE=Polytec GmbH
 
+OUI:64B21D*
+ ID_OUI_FROM_DATABASE=Chengdu Phycom Tech Co., Ltd.
+
 OUI:64B310*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
@@ -59785,6 +59977,9 @@ OUI:64B473*
 OUI:64B64A*
  ID_OUI_FROM_DATABASE=ViVOtech, Inc.
 
+OUI:64B853*
+ ID_OUI_FROM_DATABASE=Samsung Elec Co.,Ltd
+
 OUI:64B9E8*
  ID_OUI_FROM_DATABASE=Apple
 
@@ -59947,6 +60142,9 @@ OUI:6828BA*
 OUI:682DDC*
  ID_OUI_FROM_DATABASE=Wuhan Changjiang Electro-Communication Equipment CO.,LTD
 
+OUI:6836B5*
+ ID_OUI_FROM_DATABASE=DriveScale, Inc.
+
 OUI:683B1E*
  ID_OUI_FROM_DATABASE=Countwise LTD
 
@@ -60007,6 +60205,9 @@ OUI:686E48*
 OUI:687251*
  ID_OUI_FROM_DATABASE=Ubiquiti Networks
 
+OUI:6872DC*
+ ID_OUI_FROM_DATABASE=CETORY.TV Company Limited
+
 OUI:68764F*
  ID_OUI_FROM_DATABASE=Sony Mobile Communications AB
 
@@ -60070,6 +60271,9 @@ OUI:68974B*
 OUI:6897E8*
  ID_OUI_FROM_DATABASE=Society of Motion Picture &amp; Television Engineers
 
+OUI:6899CD*
+ ID_OUI_FROM_DATABASE=Cisco
+
 OUI:689C5E*
  ID_OUI_FROM_DATABASE=AcSiP Technology Corp.
 
@@ -60097,6 +60301,9 @@ OUI:68AAD2*
 OUI:68AB8A*
  ID_OUI_FROM_DATABASE=RF IDeas
 
+OUI:68AE20*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:68AF13*
  ID_OUI_FROM_DATABASE=Futura Mobility
 
@@ -60184,6 +60391,9 @@ OUI:68F06D*
 OUI:68F125*
  ID_OUI_FROM_DATABASE=Data Controls Inc.
 
+OUI:68F728*
+ ID_OUI_FROM_DATABASE=LCFC(HeFei) Electronics Technology co., ltd
+
 OUI:68F895*
  ID_OUI_FROM_DATABASE=Redflow Limited
 
@@ -60202,6 +60412,9 @@ OUI:6C0460*
 OUI:6C09D6*
  ID_OUI_FROM_DATABASE=Digiquest Electronics LTD
 
+OUI:6C0B84*
+ ID_OUI_FROM_DATABASE=Universal Global Scientific Industrial Co.,Ltd.
+
 OUI:6C0E0D*
  ID_OUI_FROM_DATABASE=Sony Ericsson Mobile Communications AB
 
@@ -60268,6 +60481,9 @@ OUI:6C3E6D*
 OUI:6C3E9C*
  ID_OUI_FROM_DATABASE=KE Knestel Elektronik GmbH
 
+OUI:6C4008*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:6C40C6*
  ID_OUI_FROM_DATABASE=Nimbus Data Systems, Inc.
 
@@ -60850,6 +61066,12 @@ OUI:70F927*
 OUI:70F96D*
  ID_OUI_FROM_DATABASE=Hangzhou H3C Technologies Co., Limited
 
+OUI:70FC8C*
+ ID_OUI_FROM_DATABASE=OneAccess SA
+
+OUI:70FF5C*
+ ID_OUI_FROM_DATABASE=Cheerzing Communication(Xiamen)Technology Co.,Ltd
+
 OUI:70FF76*
  ID_OUI_FROM_DATABASE=Texas Instruments
 
@@ -60928,12 +61150,18 @@ OUI:744D79*
 OUI:745327*
  ID_OUI_FROM_DATABASE=COMMSEN CO., LIMITED
 
+OUI:74547D*
+ ID_OUI_FROM_DATABASE=Cisco SPVTG
+
 OUI:745612*
  ID_OUI_FROM_DATABASE=ARRIS Group, Inc.
 
 OUI:745798*
  ID_OUI_FROM_DATABASE=TRUMPF Laser GmbH + Co. KG
 
+OUI:745C9F*
+ ID_OUI_FROM_DATABASE=TCT mobile ltd.
+
 OUI:745E1C*
  ID_OUI_FROM_DATABASE=PIONEER CORPORATION
 
@@ -60967,6 +61195,9 @@ OUI:746F3D*
 OUI:7472F2*
  ID_OUI_FROM_DATABASE=Chipsip Technology Co., Ltd.
 
+OUI:747548*
+ ID_OUI_FROM_DATABASE=Amazon Technologies Inc.
+
 OUI:747818*
  ID_OUI_FROM_DATABASE=ServiceAssure
 
@@ -61117,6 +61348,9 @@ OUI:74E537*
 OUI:74E543*
  ID_OUI_FROM_DATABASE=Liteon Technology Corporation
 
+OUI:74E6E2*
+ ID_OUI_FROM_DATABASE=Dell Inc.
+
 OUI:74E7C6*
  ID_OUI_FROM_DATABASE=ARRIS Group, Inc.
 
@@ -61921,6 +62155,9 @@ OUI:8005DF*
 OUI:8007A2*
  ID_OUI_FROM_DATABASE=Esson Technology Inc.
 
+OUI:800902*
+ ID_OUI_FROM_DATABASE=Keysight Technologies, Inc.
+
 OUI:800A06*
  ID_OUI_FROM_DATABASE=COMTEC co.,ltd
 
@@ -61945,6 +62182,9 @@ OUI:8018A7*
 OUI:801934*
  ID_OUI_FROM_DATABASE=Intel Corporate
 
+OUI:801967*
+ ID_OUI_FROM_DATABASE=Shanghai Reallytek Information Technology  Co.,Ltd
+
 OUI:801DAA*
  ID_OUI_FROM_DATABASE=Avaya Inc
 
@@ -61972,6 +62212,9 @@ OUI:802FDE*
 OUI:803457*
  ID_OUI_FROM_DATABASE=OT Systems Limited
 
+OUI:803773*
+ ID_OUI_FROM_DATABASE=Netgear Inc
+
 OUI:8038FD*
  ID_OUI_FROM_DATABASE=LeapFrog Enterprises, Inc.
 
@@ -62146,6 +62389,9 @@ OUI:80BAE6*
 OUI:80BBEB*
  ID_OUI_FROM_DATABASE=Satmap Systems Ltd
 
+OUI:80BE05*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:80C16E*
  ID_OUI_FROM_DATABASE=Hewlett Packard
 
@@ -62236,6 +62482,9 @@ OUI:841715*
 OUI:841766*
  ID_OUI_FROM_DATABASE=Weifang GoerTek Electronics Co., Ltd
 
+OUI:841826*
+ ID_OUI_FROM_DATABASE=Osram GmbH
+
 OUI:84183A*
  ID_OUI_FROM_DATABASE=Ruckus Wireless
 
@@ -62272,6 +62521,9 @@ OUI:842615*
 OUI:84262B*
  ID_OUI_FROM_DATABASE=Alcatel-Lucent
 
+OUI:842690*
+ ID_OUI_FROM_DATABASE=BEIJING THOUGHT SCIENCE CO.,LTD.
+
 OUI:8427CE*
  ID_OUI_FROM_DATABASE=Corporation of the Presiding Bishop of The Church of Jesus Christ of Latter-day Saints
 
@@ -62332,6 +62584,9 @@ OUI:844F03*
 OUI:845181*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
+OUI:8455A5*
+ ID_OUI_FROM_DATABASE=Samsung Elec Co.,Ltd
+
 OUI:84569C*
  ID_OUI_FROM_DATABASE=Coho Data, Inc.,
 
@@ -62365,6 +62620,9 @@ OUI:84742A*
 OUI:847616*
  ID_OUI_FROM_DATABASE=Addat S.r.o.
 
+OUI:84788B*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:8478AC*
  ID_OUI_FROM_DATABASE=Cisco
 
@@ -62434,6 +62692,9 @@ OUI:849CA6*
 OUI:849DC5*
  ID_OUI_FROM_DATABASE=Centera Photonics Inc.
 
+OUI:84A466*
+ ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
+
 OUI:84A6C8*
  ID_OUI_FROM_DATABASE=Intel Corporate
 
@@ -62506,6 +62767,9 @@ OUI:84E714*
 OUI:84EA99*
  ID_OUI_FROM_DATABASE=Vieworks
 
+OUI:84EB18*
+ ID_OUI_FROM_DATABASE=Texas Instruments
+
 OUI:84ED33*
  ID_OUI_FROM_DATABASE=BBMC Co.,Ltd
 
@@ -62641,6 +62905,12 @@ OUI:88685C*
 OUI:886B76*
  ID_OUI_FROM_DATABASE=CHINA HOPEFUL GROUP HOPEFUL ELECTRIC CO.,LTD
 
+OUI:88708C*
+ ID_OUI_FROM_DATABASE=Lenovo Mobile Communication Technology Ltd.
+
+OUI:8870EF*
+ ID_OUI_FROM_DATABASE=SC Professional Trading Co., Ltd.
+
 OUI:887398*
  ID_OUI_FROM_DATABASE=K2E Tekpoint
 
@@ -62746,6 +63016,9 @@ OUI:88C626*
 OUI:88C663*
  ID_OUI_FROM_DATABASE=Apple
 
+OUI:88C9D0*
+ ID_OUI_FROM_DATABASE=LG Electronics
+
 OUI:88CB87*
  ID_OUI_FROM_DATABASE=Apple
 
@@ -62812,6 +63085,9 @@ OUI:8C006D*
 OUI:8C04FF*
  ID_OUI_FROM_DATABASE=Technicolor USA Inc.
 
+OUI:8C0551*
+ ID_OUI_FROM_DATABASE=Koubachi AG
+
 OUI:8C078C*
  ID_OUI_FROM_DATABASE=FLOW DATA INC
 
@@ -62995,6 +63271,9 @@ OUI:8C8E76*
 OUI:8C90D3*
  ID_OUI_FROM_DATABASE=Alcatel Lucent
 
+OUI:8C9109*
+ ID_OUI_FROM_DATABASE=Toyoshima Electric Technoeogy(Suzhou) Co.,Ltd.
+
 OUI:8C9236*
  ID_OUI_FROM_DATABASE=Aus.Linx Technology Co., Ltd.
 
@@ -63082,6 +63361,9 @@ OUI:8CDE52*
 OUI:8CDE99*
  ID_OUI_FROM_DATABASE=Comlab Inc.
 
+OUI:8CDF9D*
+ ID_OUI_FROM_DATABASE=NEC Corporation
+
 OUI:8CE081*
  ID_OUI_FROM_DATABASE=zte corporation
 
@@ -63133,6 +63415,12 @@ OUI:900D66*
 OUI:900DCB*
  ID_OUI_FROM_DATABASE=ARRIS Group, Inc.
 
+OUI:90179B*
+ ID_OUI_FROM_DATABASE=Nanomegas
+
+OUI:9017AC*
+ ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
+
 OUI:90185E*
  ID_OUI_FROM_DATABASE=Apex Tool Group GmbH & Co OHG
 
@@ -63166,12 +63454,18 @@ OUI:902083*
 OUI:902155*
  ID_OUI_FROM_DATABASE=HTC Corporation
 
+OUI:902181*
+ ID_OUI_FROM_DATABASE=Shanghai Huaqin Telecom Technology Co.,Ltd
+
 OUI:9027E4*
  ID_OUI_FROM_DATABASE=Apple
 
 OUI:902B34*
  ID_OUI_FROM_DATABASE=GIGA-BYTE TECHNOLOGY CO.,LTD.
 
+OUI:902CC7*
+ ID_OUI_FROM_DATABASE=C-MAX Asia Limited
+
 OUI:902E87*
  ID_OUI_FROM_DATABASE=LabJack
 
@@ -63304,6 +63598,9 @@ OUI:90840D*
 OUI:9088A2*
  ID_OUI_FROM_DATABASE=IONICS TECHNOLOGY ME LTDA
 
+OUI:908C09*
+ ID_OUI_FROM_DATABASE=Total Phase
+
 OUI:908C44*
  ID_OUI_FROM_DATABASE=H.K ZONGMU TECHNOLOGY CO., LTD.
 
@@ -63337,6 +63634,9 @@ OUI:909916*
 OUI:909DE0*
  ID_OUI_FROM_DATABASE=Newland Design + Assoc. Inc.
 
+OUI:909F33*
+ ID_OUI_FROM_DATABASE=EFM Networks
+
 OUI:909F43*
  ID_OUI_FROM_DATABASE=Accutron Instruments Inc.
 
@@ -63416,7 +63716,7 @@ OUI:90DA4E*
  ID_OUI_FROM_DATABASE=AVANU
 
 OUI:90DA6A*
- ID_OUI_FROM_DATABASE=MCC System Co., Ltd.
+ ID_OUI_FROM_DATABASE=FOCUS H&S Co., Ltd.
 
 OUI:90DB46*
  ID_OUI_FROM_DATABASE=E-LEAD ELECTRONIC CO., LTD
@@ -63580,6 +63880,9 @@ OUI:946269*
 OUI:9463D1*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
+OUI:9470D2*
+ ID_OUI_FROM_DATABASE=WINFIRM TECHNOLOGY
+
 OUI:9471AC*
  ID_OUI_FROM_DATABASE=TCT Mobile Limited
 
@@ -63646,6 +63949,9 @@ OUI:94AE61*
 OUI:94AEE3*
  ID_OUI_FROM_DATABASE=Belden Hirschmann Industries (Suzhou) Ltd.
 
+OUI:94B40F*
+ ID_OUI_FROM_DATABASE=Aruba Networks
+
 OUI:94B8C5*
  ID_OUI_FROM_DATABASE=RuggedCom Inc.
 
@@ -63661,9 +63967,15 @@ OUI:94BA56*
 OUI:94BF1E*
  ID_OUI_FROM_DATABASE=eflow Inc. / Smart Device Planning and Development Division
 
+OUI:94BF95*
+ ID_OUI_FROM_DATABASE=Shenzhen Coship Electronics Co., Ltd
+
 OUI:94C014*
  ID_OUI_FROM_DATABASE=Sorter Sp. j. Konrad Grzeszczyk MichaA, Ziomek
 
+OUI:94C038*
+ ID_OUI_FROM_DATABASE=Tallac Networks
+
 OUI:94C150*
  ID_OUI_FROM_DATABASE=2Wire Inc
 
@@ -63793,6 +64105,9 @@ OUI:980D2E*
 OUI:981094*
  ID_OUI_FROM_DATABASE=Shenzhen Vsun communication technology Co.,ltd
 
+OUI:9816EC*
+ ID_OUI_FROM_DATABASE=IC Intracom
+
 OUI:98208E*
  ID_OUI_FROM_DATABASE=Definium Technologies
 
@@ -64027,6 +64342,9 @@ OUI:98F8C1*
 OUI:98F8DB*
  ID_OUI_FROM_DATABASE=Marini Impianti Industriali s.r.l.
 
+OUI:98FAE3*
+ ID_OUI_FROM_DATABASE=Xiaomi inc.
+
 OUI:98FB12*
  ID_OUI_FROM_DATABASE=Grand Electronics (HK) Ltd
 
@@ -64582,6 +64900,9 @@ OUI:A0A130*
 OUI:A0A23C*
  ID_OUI_FROM_DATABASE=GPMS
 
+OUI:A0A3E2*
+ ID_OUI_FROM_DATABASE=Actiontec Electronics, Inc
+
 OUI:A0A763*
  ID_OUI_FROM_DATABASE=Polytron Vertrieb GmbH
 
@@ -64597,6 +64918,9 @@ OUI:A0B100*
 OUI:A0B3CC*
  ID_OUI_FROM_DATABASE=Hewlett Packard
 
+OUI:A0B4A5*
+ ID_OUI_FROM_DATABASE=Samsung Elec Co.,Ltd
+
 OUI:A0B5DA*
  ID_OUI_FROM_DATABASE=HongKong THTF Co., Ltd
 
@@ -64724,7 +65048,7 @@ OUI:A40CC3*
  ID_OUI_FROM_DATABASE=CISCO SYSTEMS, INC.
 
 OUI:A41242*
- ID_OUI_FROM_DATABASE=NEC AccessTechnica, Ltd.
+ ID_OUI_FROM_DATABASE=NEC Platforms, Ltd.
 
 OUI:A4134E*
  ID_OUI_FROM_DATABASE=Luxul
@@ -64753,6 +65077,9 @@ OUI:A42305*
 OUI:A424B3*
  ID_OUI_FROM_DATABASE=FlatFrog Laboratories AB
 
+OUI:A4251B*
+ ID_OUI_FROM_DATABASE=Avaya, Inc
+
 OUI:A42940*
  ID_OUI_FROM_DATABASE=Shenzhen YOUHUA Technology Co., Ltd
 
@@ -64783,6 +65110,9 @@ OUI:A4466B*
 OUI:A446FA*
  ID_OUI_FROM_DATABASE=AmTRAN Video Corporation
 
+OUI:A44AD3*
+ ID_OUI_FROM_DATABASE=ST Electronics(Shanghai) Co.,Ltd
+
 OUI:A44B15*
  ID_OUI_FROM_DATABASE=Sun Cupid Technology (HK) LTD
 
@@ -64888,6 +65218,9 @@ OUI:A49A58*
 OUI:A49B13*
  ID_OUI_FROM_DATABASE=Burroughs Payment Systems, Inc.
 
+OUI:A49D49*
+ ID_OUI_FROM_DATABASE=Ketra, Inc.
+
 OUI:A49EDB*
  ID_OUI_FROM_DATABASE=AutoCrib, Inc.
 
@@ -64897,6 +65230,9 @@ OUI:A49F85*
 OUI:A49F89*
  ID_OUI_FROM_DATABASE=Shanghai Rui Rui Communication Technology Co.Ltd.
 
+OUI:A4A1C2*
+ ID_OUI_FROM_DATABASE=Ericsson AB (EAB)
+
 OUI:A4A24A*
  ID_OUI_FROM_DATABASE=Cisco SPVTG
 
@@ -65158,6 +65494,9 @@ OUI:A8776F*
 OUI:A87B39*
  ID_OUI_FROM_DATABASE=Nokia Corporation
 
+OUI:A87C01*
+ ID_OUI_FROM_DATABASE=Samsung Elec Co.,Ltd
+
 OUI:A87E33*
  ID_OUI_FROM_DATABASE=Nokia Danmark A/S
 
@@ -65209,6 +65548,9 @@ OUI:A8995C*
 OUI:A89B10*
  ID_OUI_FROM_DATABASE=inMotion Ltd.
 
+OUI:A89DD2*
+ ID_OUI_FROM_DATABASE=Shanghai DareGlobal Technologies Co., Ltd
+
 OUI:A8A668*
  ID_OUI_FROM_DATABASE=zte corporation
 
@@ -65245,6 +65587,9 @@ OUI:A8CCC5*
 OUI:A8CE90*
  ID_OUI_FROM_DATABASE=CVC
 
+OUI:A8D0E3*
+ ID_OUI_FROM_DATABASE=Systech Electronics Ltd.
+
 OUI:A8D0E5*
  ID_OUI_FROM_DATABASE=Juniper Networks
 
@@ -65254,6 +65599,9 @@ OUI:A8D236*
 OUI:A8D3C8*
  ID_OUI_FROM_DATABASE=Wachendorff Elektronik  GmbH & Co. KG
 
+OUI:A8D88A*
+ ID_OUI_FROM_DATABASE=Wyconn
+
 OUI:A8E018*
  ID_OUI_FROM_DATABASE=Nokia Corporation
 
@@ -65368,6 +65716,9 @@ OUI:AC3613*
 OUI:AC3870*
  ID_OUI_FROM_DATABASE=Lenovo Mobile Communication Technology Ltd.
 
+OUI:AC3A7A*
+ ID_OUI_FROM_DATABASE=Roku
+
 OUI:AC3C0B*
  ID_OUI_FROM_DATABASE=Apple
 
@@ -65488,6 +65839,9 @@ OUI:AC8674*
 OUI:AC867E*
  ID_OUI_FROM_DATABASE=Create New Technology (HK) Limited Company
 
+OUI:AC87A3*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:AC8ACD*
  ID_OUI_FROM_DATABASE=ROGER D.Wensker, G.Wensker sp.j.
 
@@ -65650,6 +66004,9 @@ OUI:ACF7F3*
 OUI:ACF97E*
  ID_OUI_FROM_DATABASE=ELESYS INC.
 
+OUI:ACFDCE*
+ ID_OUI_FROM_DATABASE=Intel Corporate
+
 OUI:ACFDEC*
  ID_OUI_FROM_DATABASE=Apple, Inc
 
@@ -65809,6 +66166,9 @@ OUI:B09074*
 OUI:B09134*
  ID_OUI_FROM_DATABASE=Taleo
 
+OUI:B09137*
+ ID_OUI_FROM_DATABASE=ISis ImageStream Internet Solutions, Inc
+
 OUI:B0973A*
  ID_OUI_FROM_DATABASE=E-Fuel Corporation
 
@@ -66031,6 +66391,9 @@ OUI:B435F7*
 OUI:B43741*
  ID_OUI_FROM_DATABASE=Consert, Inc.
 
+OUI:B43934*
+ ID_OUI_FROM_DATABASE=Pen Generations, Inc.
+
 OUI:B439D6*
  ID_OUI_FROM_DATABASE=ProCurve Networking by HP
 
@@ -66157,6 +66520,9 @@ OUI:B4A4E3*
 OUI:B4A5A9*
  ID_OUI_FROM_DATABASE=MODI GmbH
 
+OUI:B4A828*
+ ID_OUI_FROM_DATABASE=Shenzhen Concox Information Technology Co., Ltd
+
 OUI:B4A82B*
  ID_OUI_FROM_DATABASE=Histar Digital Electronics Co., Ltd.
 
@@ -66496,6 +66862,9 @@ OUI:B8AD3E*
 OUI:B8AE6E*
  ID_OUI_FROM_DATABASE=Nintendo Co., Ltd.
 
+OUI:B8AEED*
+ ID_OUI_FROM_DATABASE=Elitegroup Computer Systems Co., Ltd.
+
 OUI:B8AF67*
  ID_OUI_FROM_DATABASE=Hewlett-Packard Company
 
@@ -66562,6 +66931,9 @@ OUI:B8D06F*
 OUI:B8D49D*
  ID_OUI_FROM_DATABASE=M Seven System Ltd.
 
+OUI:B8D812*
+ ID_OUI_FROM_DATABASE=IEEE REGISTRATION AUTHORITY  - Please see MAM public listing for more information.
+
 OUI:B8D9CE*
  ID_OUI_FROM_DATABASE=Samsung Electronics
 
@@ -66658,6 +67030,9 @@ OUI:BC15A6*
 OUI:BC1665*
  ID_OUI_FROM_DATABASE=Cisco
 
+OUI:BC16F5*
+ ID_OUI_FROM_DATABASE=Cisco
+
 OUI:BC1A67*
  ID_OUI_FROM_DATABASE=YF Technology Co., Ltd
 
@@ -66766,6 +67141,9 @@ OUI:BC6A16*
 OUI:BC6A29*
  ID_OUI_FROM_DATABASE=Texas Instruments
 
+OUI:BC6B4D*
+ ID_OUI_FROM_DATABASE=Alcatel-Lucent
+
 OUI:BC6E76*
  ID_OUI_FROM_DATABASE=Green Energy Options Ltd
 
@@ -66862,12 +67240,18 @@ OUI:BCBAE1*
 OUI:BCBBC9*
  ID_OUI_FROM_DATABASE=Kellendonk Elektronik GmbH
 
+OUI:BCBC46*
+ ID_OUI_FROM_DATABASE=SKS Welding Systems GmbH
+
 OUI:BCC168*
  ID_OUI_FROM_DATABASE=DinBox Sverige AB
 
 OUI:BCC23A*
  ID_OUI_FROM_DATABASE=Thomson Video Networks
 
+OUI:BCC342*
+ ID_OUI_FROM_DATABASE=Panasonic System Networks Co., Ltd.
+
 OUI:BCC61A*
  ID_OUI_FROM_DATABASE=SPECTRA EMBEDDED SYSTEMS
 
@@ -66907,6 +67291,9 @@ OUI:BCE59F*
 OUI:BCEA2B*
  ID_OUI_FROM_DATABASE=CityCom GmbH
 
+OUI:BCEAFA*
+ ID_OUI_FROM_DATABASE=Hewlett Packard
+
 OUI:BCEE7B*
  ID_OUI_FROM_DATABASE=ASUSTek COMPUTER INC.
 
@@ -66976,6 +67363,9 @@ OUI:C03580*
 OUI:C035BD*
  ID_OUI_FROM_DATABASE=Velocytech Aps
 
+OUI:C03896*
+ ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
+
 OUI:C038F9*
  ID_OUI_FROM_DATABASE=Nokia Danmark A/S
 
@@ -67087,6 +67477,9 @@ OUI:C09132*
 OUI:C09134*
  ID_OUI_FROM_DATABASE=ProCurve Networking by HP
 
+OUI:C09879*
+ ID_OUI_FROM_DATABASE=Acer Inc.
+
 OUI:C098E5*
  ID_OUI_FROM_DATABASE=University of Michigan
 
@@ -67228,6 +67621,9 @@ OUI:C40528*
 OUI:C4084A*
  ID_OUI_FROM_DATABASE=Alcatel-Lucent
 
+OUI:C40880*
+ ID_OUI_FROM_DATABASE=Shenzhen UTEPO Tech Co., Ltd.
+
 OUI:C40938*
  ID_OUI_FROM_DATABASE=Fujian Star-net Communication Co., Ltd
 
@@ -67573,6 +67969,9 @@ OUI:C819F7*
 OUI:C81AFE*
  ID_OUI_FROM_DATABASE=DLOGIC GmbH
 
+OUI:C81B6B*
+ ID_OUI_FROM_DATABASE=Innova Security
+
 OUI:C81E8E*
  ID_OUI_FROM_DATABASE=ADV Security (S) Pte Ltd
 
@@ -67675,6 +68074,9 @@ OUI:C88439*
 OUI:C88447*
  ID_OUI_FROM_DATABASE=Beautiful Enterprise Co., Ltd
 
+OUI:C88550*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:C8873B*
  ID_OUI_FROM_DATABASE=Net Optics
 
@@ -67774,6 +68176,9 @@ OUI:C8CBB8*
 OUI:C8CD72*
  ID_OUI_FROM_DATABASE=SAGEMCOM
 
+OUI:C8D019*
+ ID_OUI_FROM_DATABASE=Shanghai Tigercel Communication Technology Co.,Ltd
+
 OUI:C8D10B*
  ID_OUI_FROM_DATABASE=Nokia Corporation
 
@@ -67960,6 +68365,9 @@ OUI:CC3D82*
 OUI:CC3E5F*
  ID_OUI_FROM_DATABASE=Hewlett Packard
 
+OUI:CC3F1D*
+ ID_OUI_FROM_DATABASE=Intesis Software SL
+
 OUI:CC43E3*
  ID_OUI_FROM_DATABASE=Trump s.a.
 
@@ -68254,6 +68662,9 @@ OUI:D01AA7*
 OUI:D01CBB*
  ID_OUI_FROM_DATABASE=Beijing Ctimes Digital Technology Co., Ltd.
 
+OUI:D02212*
+ ID_OUI_FROM_DATABASE=IEEE REGISTRATION AUTHORITY  - Please see MAM public listing for more information.
+
 OUI:D022BE*
  ID_OUI_FROM_DATABASE=Samsung Electro Mechanics co.,LTD.
 
@@ -68668,6 +69079,9 @@ OUI:D45C70*
 OUI:D45D42*
  ID_OUI_FROM_DATABASE=Nokia Corporation
 
+OUI:D46132*
+ ID_OUI_FROM_DATABASE=Pro Concept Manufacturer Co.,Ltd.
+
 OUI:D464F7*
  ID_OUI_FROM_DATABASE=CHENGDU USEE DIGITAL TECHNOLOGY CO., LTD
 
@@ -68728,6 +69142,9 @@ OUI:D48890*
 OUI:D48CB5*
  ID_OUI_FROM_DATABASE=CISCO SYSTEMS, INC.
 
+OUI:D48DD9*
+ ID_OUI_FROM_DATABASE=Meld Technology, Inc
+
 OUI:D48F33*
  ID_OUI_FROM_DATABASE=Microsoft Corporation
 
@@ -68887,6 +69304,9 @@ OUI:D4F0B4*
 OUI:D4F143*
  ID_OUI_FROM_DATABASE=IPROAD.,Inc
 
+OUI:D4F46F*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:D4F63F*
  ID_OUI_FROM_DATABASE=IEA S.R.L.
 
@@ -68974,6 +69394,9 @@ OUI:D831CF*
 OUI:D8337F*
  ID_OUI_FROM_DATABASE=Office FA.com Co.,Ltd.
 
+OUI:D83C69*
+ ID_OUI_FROM_DATABASE=Tinno Mobile Technology Corp
+
 OUI:D842AC*
  ID_OUI_FROM_DATABASE=Shanghai Feixun Communication Co.,Ltd.
 
@@ -69058,9 +69481,15 @@ OUI:D87CDD*
 OUI:D87EB1*
  ID_OUI_FROM_DATABASE=x.o.ware, inc.
 
+OUI:D88039*
+ ID_OUI_FROM_DATABASE=Microchip Technology Inc.
+
 OUI:D881CE*
  ID_OUI_FROM_DATABASE=AHN INC.
 
+OUI:D88466*
+ ID_OUI_FROM_DATABASE=Extreme Networks
+
 OUI:D88A3B*
  ID_OUI_FROM_DATABASE=UNIT-EM
 
@@ -69088,6 +69517,9 @@ OUI:D89760*
 OUI:D8977C*
  ID_OUI_FROM_DATABASE=Grey Innovation
 
+OUI:D897BA*
+ ID_OUI_FROM_DATABASE=PEGATRON CORPORATION
+
 OUI:D89D67*
  ID_OUI_FROM_DATABASE=Hewlett Packard
 
@@ -69121,6 +69553,9 @@ OUI:D8B12A*
 OUI:D8B377*
  ID_OUI_FROM_DATABASE=HTC Corporation
 
+OUI:D8B6B7*
+ ID_OUI_FROM_DATABASE=Comtrend Corporation
+
 OUI:D8B6C1*
  ID_OUI_FROM_DATABASE=NetworkAccountant, Inc.
 
@@ -69151,6 +69586,9 @@ OUI:D8C7C8*
 OUI:D8C99D*
  ID_OUI_FROM_DATABASE=EA DISPLAY LIMITED
 
+OUI:D8CB8A*
+ ID_OUI_FROM_DATABASE=Micro-Star INTL CO., LTD.
+
 OUI:D8CF9C*
  ID_OUI_FROM_DATABASE=Apple
 
@@ -69214,6 +69652,12 @@ OUI:D8EE78*
 OUI:D8F0F2*
  ID_OUI_FROM_DATABASE=Zeebo Inc
 
+OUI:D8F710*
+ ID_OUI_FROM_DATABASE=Libre Wireless Technologies Inc.
+
+OUI:D8FB11*
+ ID_OUI_FROM_DATABASE=AXACORE
+
 OUI:D8FC93*
  ID_OUI_FROM_DATABASE=Intel Corporate
 
@@ -69286,6 +69730,9 @@ OUI:DC2C26*
 OUI:DC2E6A*
  ID_OUI_FROM_DATABASE=HCT. Co., Ltd.
 
+OUI:DC2F03*
+ ID_OUI_FROM_DATABASE=Step forward Group Co., Ltd.
+
 OUI:DC309C*
  ID_OUI_FROM_DATABASE=Heyrex Limited
 
@@ -69418,6 +69865,9 @@ OUI:DCC101*
 OUI:DCC422*
  ID_OUI_FROM_DATABASE=Systembase Limited
 
+OUI:DCC622*
+ ID_OUI_FROM_DATABASE=BUHEUNG SYSTEM
+
 OUI:DCC793*
  ID_OUI_FROM_DATABASE=Nokia Corporation
 
@@ -69448,6 +69898,9 @@ OUI:DCD52A*
 OUI:DCD87F*
  ID_OUI_FROM_DATABASE=Shenzhen JoinCyber Telecom Equipment Ltd
 
+OUI:DCDA4F*
+ ID_OUI_FROM_DATABASE=GETCK TECHNOLOGY,  INC
+
 OUI:DCDECA*
  ID_OUI_FROM_DATABASE=Akyllor
 
@@ -69460,6 +69913,9 @@ OUI:DCE578*
 OUI:DCE71C*
  ID_OUI_FROM_DATABASE=AUG Elektronik GmbH
 
+OUI:DCEC06*
+ ID_OUI_FROM_DATABASE=Heimi Network Technology Co., Ltd.
+
 OUI:DCF05D*
  ID_OUI_FROM_DATABASE=Letta Teknoloji
 
@@ -69490,6 +69946,9 @@ OUI:E00B28*
 OUI:E00C7F*
  ID_OUI_FROM_DATABASE=Nintendo Co., Ltd.
 
+OUI:E0107F*
+ ID_OUI_FROM_DATABASE=Ruckus Wireless
+
 OUI:E0143E*
  ID_OUI_FROM_DATABASE=Modoosis Inc.
 
@@ -69550,6 +70009,9 @@ OUI:E039D7*
 OUI:E03C5B*
  ID_OUI_FROM_DATABASE=SHENZHEN JIAXINJIE ELECTRON CO.,LTD
 
+OUI:E03E44*
+ ID_OUI_FROM_DATABASE=Broadcom Corporation
+
 OUI:E03E4A*
  ID_OUI_FROM_DATABASE=Cavanagh Group International
 
@@ -69688,6 +70150,9 @@ OUI:E0AF4B*
 OUI:E0B2F1*
  ID_OUI_FROM_DATABASE=FN-LINK TECHNOLOGY LIMITED
 
+OUI:E0B52D*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:E0B7B1*
  ID_OUI_FROM_DATABASE=Pace plc
 
@@ -69829,9 +70294,15 @@ OUI:E41289*
 OUI:E41C4B*
  ID_OUI_FROM_DATABASE=V2 TECHNOLOGY, INC.
 
+OUI:E41D2D*
+ ID_OUI_FROM_DATABASE=Mellanox Technologies, Inc.
+
 OUI:E41F13*
  ID_OUI_FROM_DATABASE=IBM Corp
 
+OUI:E42354*
+ ID_OUI_FROM_DATABASE=SHENZHEN FUZHI SOFTWARE TECHNOLOGY CO.,LTD
+
 OUI:E425E7*
  ID_OUI_FROM_DATABASE=Apple
 
@@ -69961,6 +70432,9 @@ OUI:E48AD5*
 OUI:E48B7F*
  ID_OUI_FROM_DATABASE=Apple
 
+OUI:E48C0F*
+ ID_OUI_FROM_DATABASE=Discovery Insure
+
 OUI:E49069*
  ID_OUI_FROM_DATABASE=Rockwell Automation
 
@@ -70060,6 +70534,9 @@ OUI:E4F4C6*
 OUI:E4F7A1*
  ID_OUI_FROM_DATABASE=Datafox GmbH
 
+OUI:E4F8EF*
+ ID_OUI_FROM_DATABASE=Samsung Elec Co.,Ltd
+
 OUI:E4FA1D*
  ID_OUI_FROM_DATABASE=PAD Peripheral Advanced Design Inc.
 
@@ -70111,6 +70588,9 @@ OUI:E8150E*
 OUI:E817FC*
  ID_OUI_FROM_DATABASE=NIFTY Corporation
 
+OUI:E81863*
+ ID_OUI_FROM_DATABASE=IEEE REGISTRATION AUTHORITY  - Please see MAM public listing for more information.
+
 OUI:E82877*
  ID_OUI_FROM_DATABASE=TMY Co., Ltd.
 
@@ -70228,6 +70708,9 @@ OUI:E8802E*
 OUI:E880D8*
  ID_OUI_FROM_DATABASE=GNTEK Electronics Co.,Ltd.
 
+OUI:E887A3*
+ ID_OUI_FROM_DATABASE=Loxley Public Company Limited
+
 OUI:E8892C*
  ID_OUI_FROM_DATABASE=ARRIS Group, Inc.
 
@@ -70249,6 +70732,9 @@ OUI:E8944C*
 OUI:E894F6*
  ID_OUI_FROM_DATABASE=TP-LINK TECHNOLOGIES CO.,LTD.
 
+OUI:E89606*
+ ID_OUI_FROM_DATABASE=testo Instruments (Shenzhen) Co., Ltd.
+
 OUI:E8995A*
  ID_OUI_FROM_DATABASE=PiiGAB, Processinformation i Goteborg AB
 
@@ -70303,6 +70789,9 @@ OUI:E8C320*
 OUI:E8CBA1*
  ID_OUI_FROM_DATABASE=Nokia Corporation
 
+OUI:E8CC18*
+ ID_OUI_FROM_DATABASE=D-Link International
+
 OUI:E8CC32*
  ID_OUI_FROM_DATABASE=Micronet  LTD
 
@@ -70396,6 +70885,9 @@ OUI:EC0ED6*
 OUI:EC1120*
  ID_OUI_FROM_DATABASE=FloDesign Wind Turbine Corporation
 
+OUI:EC13B2*
+ ID_OUI_FROM_DATABASE=Netonix
+
 OUI:EC14F6*
  ID_OUI_FROM_DATABASE=BioControl AS
 
@@ -70681,6 +71173,9 @@ OUI:F015A0*
 OUI:F01C13*
  ID_OUI_FROM_DATABASE=LG Electronics
 
+OUI:F01C2D*
+ ID_OUI_FROM_DATABASE=Juniper Networks
+
 OUI:F01FAF*
  ID_OUI_FROM_DATABASE=Dell Inc
 
@@ -70729,6 +71224,9 @@ OUI:F03A4B*
 OUI:F03A55*
  ID_OUI_FROM_DATABASE=Omega Elektronik AS
 
+OUI:F03D29*
+ ID_OUI_FROM_DATABASE=Actility
+
 OUI:F03FF8*
  ID_OUI_FROM_DATABASE=R L Drake
 
@@ -70801,6 +71299,9 @@ OUI:F07BCB*
 OUI:F07D68*
  ID_OUI_FROM_DATABASE=D-Link Corporation
 
+OUI:F07F06*
+ ID_OUI_FROM_DATABASE=Cisco
+
 OUI:F07F0C*
  ID_OUI_FROM_DATABASE=Leopold Kostal GmbH &Co. KG
 
@@ -70843,6 +71344,9 @@ OUI:F09CBB*
 OUI:F09CE9*
  ID_OUI_FROM_DATABASE=Aerohive Networks Inc
 
+OUI:F09E63*
+ ID_OUI_FROM_DATABASE=Cisco
+
 OUI:F0A764*
  ID_OUI_FROM_DATABASE=GST Co., Ltd.
 
@@ -71062,6 +71566,9 @@ OUI:F4472A*
 OUI:F44848*
  ID_OUI_FROM_DATABASE=Amscreen Group Ltd
 
+OUI:F44E05*
+ ID_OUI_FROM_DATABASE=Cisco
+
 OUI:F44EFD*
  ID_OUI_FROM_DATABASE=Actions Semiconductor Co.,Ltd.(Cayman Islands)
 
@@ -71188,6 +71695,9 @@ OUI:F4B72A*
 OUI:F4B7E2*
  ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
 
+OUI:F4B85E*
+ ID_OUI_FROM_DATABASE=Texas INstruments
+
 OUI:F4BD7C*
  ID_OUI_FROM_DATABASE=Chengdu jinshi communication Co., LTD
 
@@ -71461,6 +71971,9 @@ OUI:F87B8C*
 OUI:F8811A*
  ID_OUI_FROM_DATABASE=OVERKIZ
 
+OUI:F88479*
+ ID_OUI_FROM_DATABASE=Yaojin Technology(Shenzhen)Co.,Ltd
+
 OUI:F884F2*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
@@ -71599,6 +72112,9 @@ OUI:F8E7B5*
 OUI:F8E811*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
+OUI:F8E903*
+ ID_OUI_FROM_DATABASE=D-Link International
+
 OUI:F8E968*
  ID_OUI_FROM_DATABASE=Egker Kft.
 
@@ -71782,6 +72298,9 @@ OUI:FC5B24*
 OUI:FC5B26*
  ID_OUI_FROM_DATABASE=MikroBits
 
+OUI:FC5B39*
+ ID_OUI_FROM_DATABASE=Cisco
+
 OUI:FC6018*
  ID_OUI_FROM_DATABASE=Zhejiang Kangtai Electric Co., Ltd.
 
diff --git a/hwdb/20-pci-vendor-model.hwdb b/hwdb/20-pci-vendor-model.hwdb
index 77c727dcc0..3bcdbc0d1f 100644
--- a/hwdb/20-pci-vendor-model.hwdb
+++ b/hwdb/20-pci-vendor-model.hwdb
@@ -10478,6 +10478,9 @@ pci:v00001014d0000034Asv00001014sd000004C8*
 pci:v00001014d0000034Asv00001014sd00000C49*
  ID_MODEL_FROM_DATABASE=PCI-E IPR SAS Adapter (ASIC) (PCIe3 x 8 Cache SAS RAID Internal Adapter 6GB(2CCD))
 
+pci:v00001014d0000044B*
+ ID_MODEL_FROM_DATABASE=GenWQE Accelerator Adapter
+
 pci:v00001014d000004AA*
  ID_MODEL_FROM_DATABASE=Flash Adapter 90 (PCIe2 0.9TB)
 
@@ -26387,6 +26390,9 @@ pci:v000010DEd00000FE2*
 pci:v000010DEd00000FE3*
  ID_MODEL_FROM_DATABASE=GK107M [GeForce GT 745M]
 
+pci:v000010DEd00000FE3sv0000103Csd00002B16*
+ ID_MODEL_FROM_DATABASE=GK107M [GeForce GT 745M] (GeForce GT 745A)
+
 pci:v000010DEd00000FE3sv000017AAsd00003675*
  ID_MODEL_FROM_DATABASE=GK107M [GeForce GT 745M] (GeForce GT 745A)
 
@@ -26420,6 +26426,9 @@ pci:v000010DEd00000FF1*
 pci:v000010DEd00000FF2*
  ID_MODEL_FROM_DATABASE=GK107GL [GRID K1]
 
+pci:v000010DEd00000FF3*
+ ID_MODEL_FROM_DATABASE=GK107GL [Quadro K420]
+
 pci:v000010DEd00000FF5*
  ID_MODEL_FROM_DATABASE=GK107GL [GRID K1 Tesla USM]
 
@@ -26546,6 +26555,9 @@ pci:v000010DEd00001029*
 pci:v000010DEd0000103A*
  ID_MODEL_FROM_DATABASE=GK110GL [Quadro K6000]
 
+pci:v000010DEd0000103C*
+ ID_MODEL_FROM_DATABASE=GK110GL [Quadro K5200]
+
 pci:v000010DEd00001040*
  ID_MODEL_FROM_DATABASE=GF119 [GeForce GT 520]
 
@@ -26981,6 +26993,12 @@ pci:v000010DEd00001140sv00001028sd000005F4*
 pci:v000010DEd00001140sv00001028sd0000064E*
  ID_MODEL_FROM_DATABASE=GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M] (GeForce 820M)
 
+pci:v000010DEd00001140sv00001028sd00000652*
+ ID_MODEL_FROM_DATABASE=GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M] (GeForce 820M)
+
+pci:v000010DEd00001140sv00001028sd00000662*
+ ID_MODEL_FROM_DATABASE=GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M] (GeForce 820M)
+
 pci:v000010DEd00001140sv00001028sd0000068D*
  ID_MODEL_FROM_DATABASE=GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M] (GeForce 820M)
 
@@ -27239,9 +27257,15 @@ pci:v000010DEd00001140sv0000152Dsd00000982*
 pci:v000010DEd00001140sv0000152Dsd00000983*
  ID_MODEL_FROM_DATABASE=GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M] (GeForce GT 630M)
 
+pci:v000010DEd00001140sv0000152Dsd00001005*
+ ID_MODEL_FROM_DATABASE=GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M] (GeForce GT 820M)
+
 pci:v000010DEd00001140sv0000152Dsd00001012*
  ID_MODEL_FROM_DATABASE=GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M] (GeForce 710M)
 
+pci:v000010DEd00001140sv0000152Dsd00001019*
+ ID_MODEL_FROM_DATABASE=GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M] (GeForce 820M)
+
 pci:v000010DEd00001140sv0000152Dsd00001030*
  ID_MODEL_FROM_DATABASE=GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M] (GeForce GT 630M)
 
@@ -27254,6 +27278,9 @@ pci:v000010DEd00001140sv0000152Dsd00001067*
 pci:v000010DEd00001140sv0000152Dsd00001072*
  ID_MODEL_FROM_DATABASE=GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M] (GeForce GT 720M)
 
+pci:v000010DEd00001140sv0000152Dsd00001086*
+ ID_MODEL_FROM_DATABASE=GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M] (GeForce 820M)
+
 pci:v000010DEd00001140sv000017AAsd00002200*
  ID_MODEL_FROM_DATABASE=GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M] (NVS 5200M)
 
@@ -27578,6 +27605,9 @@ pci:v000010DEd000011B0sv000010DEsd0000101B*
 pci:v000010DEd000011B1*
  ID_MODEL_FROM_DATABASE=GK104GL [GRID K2 Tesla USM]
 
+pci:v000010DEd000011B4*
+ ID_MODEL_FROM_DATABASE=GK104GL [Quadro K4200]
+
 pci:v000010DEd000011B6*
  ID_MODEL_FROM_DATABASE=GK104GLM [Quadro K3100M]
 
@@ -27863,6 +27893,9 @@ pci:v000010DEd00001295sv0000103Csd00002B0F*
 pci:v000010DEd00001295sv0000103Csd00002B11*
  ID_MODEL_FROM_DATABASE=GK208M [GeForce 710M] (GeForce GT 710A)
 
+pci:v000010DEd00001295sv0000103Csd00002B20*
+ ID_MODEL_FROM_DATABASE=GK208M [GeForce 710M] (GeForce 810A)
+
 pci:v000010DEd00001295sv0000103Csd00002B21*
  ID_MODEL_FROM_DATABASE=GK208M [GeForce 710M] (GeForce GT 810A)
 
@@ -27887,6 +27920,9 @@ pci:v000010DEd000012BA*
 pci:v000010DEd00001340*
  ID_MODEL_FROM_DATABASE=GM108M [GeForce 830M]
 
+pci:v000010DEd00001340sv0000103Csd00002B2B*
+ ID_MODEL_FROM_DATABASE=GM108M [GeForce 830M] (GeForce 830A)
+
 pci:v000010DEd00001341*
  ID_MODEL_FROM_DATABASE=GM108M [GeForce 840M]
 
@@ -27926,6 +27962,15 @@ pci:v000010DEd00001392*
 pci:v000010DEd00001393*
  ID_MODEL_FROM_DATABASE=GM107M [GeForce 840M]
 
+pci:v000010DEd000013B3*
+ ID_MODEL_FROM_DATABASE=GM107GLM [Quadro K2200M]
+
+pci:v000010DEd000013BA*
+ ID_MODEL_FROM_DATABASE=GM107GL [Quadro K2200]
+
+pci:v000010DEd000013BB*
+ ID_MODEL_FROM_DATABASE=GM107GL [Quadro K620]
+
 pci:v000010DF*
  ID_VENDOR_FROM_DATABASE=Emulex Corporation
 
@@ -29291,6 +29336,9 @@ pci:v00001103d00000622*
 pci:v00001103d00000640*
  ID_MODEL_FROM_DATABASE=RocketRAID 640 4 Port SATA-III Controller
 
+pci:v00001103d00000644*
+ ID_MODEL_FROM_DATABASE=RocketRAID 644 4 Port SATA-III Controller (eSATA)
+
 pci:v00001103d00001720*
  ID_MODEL_FROM_DATABASE=RocketRAID 1720 (2x SATA II RAID Controller)
 
@@ -32403,11 +32451,14 @@ pci:v00001131d00007146sv0000153Bsd00001155*
  ID_MODEL_FROM_DATABASE=SAA7146 (Cinergy 1200 DVB-S)
 
 pci:v00001131d00007146sv0000153Bsd00001156*
- ID_MODEL_FROM_DATABASE=SAA7146 (Terratec Cynergy 1200C)
+ ID_MODEL_FROM_DATABASE=SAA7146 (Cinergy 1200 DVB-C)
 
 pci:v00001131d00007146sv0000153Bsd00001157*
  ID_MODEL_FROM_DATABASE=SAA7146 (Cinergy 1200 DVB-T)
 
+pci:v00001131d00007146sv0000153Bsd00001176*
+ ID_MODEL_FROM_DATABASE=SAA7146 (Cinergy 1200 DVB-C (MK3))
+
 pci:v00001131d00007146sv00001894sd00000020*
  ID_MODEL_FROM_DATABASE=SAA7146 (KNC One DVB-C V1.0)
 
@@ -40322,6 +40373,12 @@ pci:v00001360d00000205*
 pci:v00001360d00000206*
  ID_MODEL_FROM_DATABASE=GPS180PEX GPS Receiver (PCI Express)
 
+pci:v00001360d00000207*
+ ID_MODEL_FROM_DATABASE=GLN180PEX GPS/GLONASS receiver (PCI Express)
+
+pci:v00001360d00000208*
+ ID_MODEL_FROM_DATABASE=GPS180AMC GPS Receiver (PCI Express / MicroTCA / AdvancedMC)
+
 pci:v00001360d00000301*
  ID_MODEL_FROM_DATABASE=TCR510PCI IRIG Timecode Reader
 
@@ -41522,6 +41579,9 @@ pci:v000013F6d00008788sv00001043sd00008428*
 pci:v000013F6d00008788sv00001043sd00008467*
  ID_MODEL_FROM_DATABASE=CMI8788 [Oxygen HD Audio] (CMI8786 (Xonar DG))
 
+pci:v000013F6d00008788sv00001043sd000085F4*
+ ID_MODEL_FROM_DATABASE=CMI8788 [Oxygen HD Audio] (Virtuoso 100 (Xonar Essence STX II))
+
 pci:v000013F6d00008788sv000013F6sd00008782*
  ID_MODEL_FROM_DATABASE=CMI8788 [Oxygen HD Audio] (PCI 2.0 HD Audio)
 
@@ -42846,7 +42906,7 @@ pci:v00001425d00005012*
  ID_MODEL_FROM_DATABASE=T560-CR Unified Wire Ethernet Controller
 
 pci:v00001425d00005013*
- ID_MODEL_FROM_DATABASE=T580-CR Unified Wire Ethernet Controller
+ ID_MODEL_FROM_DATABASE=T580-CHR Unified Wire Ethernet Controller
 
 pci:v00001425d00005014*
  ID_MODEL_FROM_DATABASE=T580-LP-SO-CR Unified Wire Ethernet Controller
@@ -42924,7 +42984,7 @@ pci:v00001425d00005412*
  ID_MODEL_FROM_DATABASE=T560-CR Unified Wire Ethernet Controller
 
 pci:v00001425d00005413*
- ID_MODEL_FROM_DATABASE=T580-CR Unified Wire Ethernet Controller
+ ID_MODEL_FROM_DATABASE=T580-CHR Unified Wire Ethernet Controller
 
 pci:v00001425d00005414*
  ID_MODEL_FROM_DATABASE=T580-LP-SO-CR Unified Wire Ethernet Controller
@@ -43002,7 +43062,7 @@ pci:v00001425d00005512*
  ID_MODEL_FROM_DATABASE=T560-CR Unified Wire Storage Controller
 
 pci:v00001425d00005513*
- ID_MODEL_FROM_DATABASE=T580-CR Unified Wire Storage Controller
+ ID_MODEL_FROM_DATABASE=T580-CHR Unified Wire Storage Controller
 
 pci:v00001425d00005514*
  ID_MODEL_FROM_DATABASE=T580-LP-SO-CR Unified Wire Storage Controller
@@ -43080,7 +43140,7 @@ pci:v00001425d00005612*
  ID_MODEL_FROM_DATABASE=T560-CR Unified Wire Storage Controller
 
 pci:v00001425d00005613*
- ID_MODEL_FROM_DATABASE=T580-CR Unified Wire Storage Controller
+ ID_MODEL_FROM_DATABASE=T580-CHR Unified Wire Storage Controller
 
 pci:v00001425d00005614*
  ID_MODEL_FROM_DATABASE=T580-LP-SO-CR Unified Wire Storage Controller
@@ -43236,7 +43296,7 @@ pci:v00001425d00005812*
  ID_MODEL_FROM_DATABASE=T560-CR Unified Wire Ethernet Controller [VF]
 
 pci:v00001425d00005813*
- ID_MODEL_FROM_DATABASE=T580-CR Unified Wire Ethernet Controller [VF]
+ ID_MODEL_FROM_DATABASE=T580-CHR Unified Wire Ethernet Controller [VF]
 
 pci:v00001425d00005814*
  ID_MODEL_FROM_DATABASE=T580-LP-SO-CR Unified Wire Ethernet Controller [VF]
@@ -45887,6 +45947,9 @@ pci:v000014E4d00004359sv00001028sd00000011*
 pci:v000014E4d00004359sv0000103Csd0000182C*
  ID_MODEL_FROM_DATABASE=BCM43228 802.11a/b/g/n (BCM943228HM4L 802.11a/b/g/n 2x2 Wi-Fi Adapter)
 
+pci:v000014E4d00004360*
+ ID_MODEL_FROM_DATABASE=BCM4360 802.11ac Wireless Network Adapter
+
 pci:v000014E4d00004365*
  ID_MODEL_FROM_DATABASE=BCM43142 802.11b/g/n
 
@@ -45896,6 +45959,12 @@ pci:v000014E4d00004365sv00001028sd00000016*
 pci:v000014E4d000043A0*
  ID_MODEL_FROM_DATABASE=BCM4360 802.11ac Wireless Network Adapter
 
+pci:v000014E4d000043A9*
+ ID_MODEL_FROM_DATABASE=BCM43217 802.11b/g/n
+
+pci:v000014E4d000043AA*
+ ID_MODEL_FROM_DATABASE=BCM43131 802.11b/g/n
+
 pci:v000014E4d000043B1*
  ID_MODEL_FROM_DATABASE=BCM4352 802.11ac Wireless Network Adapter
 
@@ -46875,7 +46944,7 @@ pci:v000014F1d00008852*
  ID_MODEL_FROM_DATABASE=CX23885 PCI Video and Audio Decoder
 
 pci:v000014F1d00008852sv00000070sd00008010*
- ID_MODEL_FROM_DATABASE=CX23885 PCI Video and Audio Decoder (Hauppauge WinTV HVR-1400 ExpressCard)
+ ID_MODEL_FROM_DATABASE=CX23885 PCI Video and Audio Decoder (WinTV HVR-1400 ExpressCard)
 
 pci:v000014F1d00008852sv0000107Dsd00006F22*
  ID_MODEL_FROM_DATABASE=CX23885 PCI Video and Audio Decoder (WinFast PxTV1200)
@@ -46889,6 +46958,9 @@ pci:v000014F1d00008852sv0000153Bsd0000117E*
 pci:v000014F1d00008852sv000018ACsd0000DB78*
  ID_MODEL_FROM_DATABASE=CX23885 PCI Video and Audio Decoder (FusionHDTV DVB-T Dual Express)
 
+pci:v000014F1d00008852sv00004254sd00009580*
+ ID_MODEL_FROM_DATABASE=CX23885 PCI Video and Audio Decoder (T9580)
+
 pci:v000014F1d00008880*
  ID_MODEL_FROM_DATABASE=CX23887/8 PCIe Broadcast Audio and Video Decoder with 3D Comb
 
@@ -47429,6 +47501,18 @@ pci:v00001542*
 pci:v00001542d00009260*
  ID_MODEL_FROM_DATABASE=RCIM-II Real-Time Clock & Interrupt Module
 
+pci:v00001542d00009272*
+ ID_MODEL_FROM_DATABASE=Pulse Width Modulator Card
+
+pci:v00001542d00009277*
+ ID_MODEL_FROM_DATABASE=5 Volt Delta Sigma Converter Card
+
+pci:v00001542d00009278*
+ ID_MODEL_FROM_DATABASE=10 Volt Delta Sigma Converter Card
+
+pci:v00001542d00009287*
+ ID_MODEL_FROM_DATABASE=Analog Output Card
+
 pci:v00001543*
  ID_VENDOR_FROM_DATABASE=SILICON Laboratories
 
@@ -47886,7 +47970,10 @@ pci:v000015ADd00000774*
  ID_MODEL_FROM_DATABASE=USB1.1 UHCI Controller
 
 pci:v000015ADd00000778*
- ID_MODEL_FROM_DATABASE=USB3 xHCI Controller
+ ID_MODEL_FROM_DATABASE=USB3 xHCI 0.96 Controller
+
+pci:v000015ADd00000779*
+ ID_MODEL_FROM_DATABASE=USB3 xHCI 1.0 Controller
 
 pci:v000015ADd00000790*
  ID_MODEL_FROM_DATABASE=PCI bridge
@@ -49493,6 +49580,9 @@ pci:v000016B8*
 pci:v000016BE*
  ID_VENDOR_FROM_DATABASE=Creatix Polymedia GmbH
 
+pci:v000016C3*
+ ID_VENDOR_FROM_DATABASE=Synopsys, Inc.
+
 pci:v000016C6*
  ID_VENDOR_FROM_DATABASE=Micrel-Kendin
 
@@ -52490,6 +52580,9 @@ pci:v00001A29*
 pci:v00001A29d00004338*
  ID_MODEL_FROM_DATABASE=CP8 Content Processor ASIC
 
+pci:v00001A29d00004E36*
+ ID_MODEL_FROM_DATABASE=NP6 Network Processor
+
 pci:v00001A2B*
  ID_VENDOR_FROM_DATABASE=Ascom AG
 
@@ -53921,6 +54014,9 @@ pci:v0000416Cd00000100*
 pci:v0000416Cd00000200*
  ID_MODEL_FROM_DATABASE=CPC
 
+pci:v00004254*
+ ID_VENDOR_FROM_DATABASE=DVBSky
+
 pci:v00004321*
  ID_VENDOR_FROM_DATABASE=Tata Power Strategic Electronics Division
 
@@ -59606,6 +59702,12 @@ pci:v00008086d000015A2*
 pci:v00008086d000015A3*
  ID_MODEL_FROM_DATABASE=Ethernet Connection (3) I218-V
 
+pci:v00008086d000015B7*
+ ID_MODEL_FROM_DATABASE=Ethernet Connection (2) I219-V
+
+pci:v00008086d000015B8*
+ ID_MODEL_FROM_DATABASE=Ethernet Connection (2) I219-LM
+
 pci:v00008086d00001600*
  ID_MODEL_FROM_DATABASE=Broadwell-U Host Bridge -OPI
 
@@ -69599,6 +69701,9 @@ pci:v00008086d00003B23*
 pci:v00008086d00003B25*
  ID_MODEL_FROM_DATABASE=5 Series/3400 Series Chipset SATA RAID Controller
 
+pci:v00008086d00003B25sv0000103Csd00003118*
+ ID_MODEL_FROM_DATABASE=5 Series/3400 Series Chipset SATA RAID Controller (HP Smart Array B110i SATA RAID Controller)
+
 pci:v00008086d00003B26*
  ID_MODEL_FROM_DATABASE=5 Series/3400 Series Chipset 2 port SATA IDE Controller
 
@@ -70703,6 +70808,489 @@ pci:v00008086d000065FA*
 pci:v00008086d000065FF*
  ID_MODEL_FROM_DATABASE=5100 Chipset DMA Engine
 
+pci:v00008086d00006F00*
+ ID_MODEL_FROM_DATABASE=Broadwell DMI2
+
+pci:v00008086d00006F01*
+ ID_MODEL_FROM_DATABASE=Broadwell PCI Express Root Port 0
+
+pci:v00008086d00006F02*
+ ID_MODEL_FROM_DATABASE=Broadwell PCI Express Root Port 1
+
+pci:v00008086d00006F03*
+ ID_MODEL_FROM_DATABASE=Broadwell PCI Express Root Port 1
+
+pci:v00008086d00006F04*
+ ID_MODEL_FROM_DATABASE=Broadwell PCI Express Root Port 2
+
+pci:v00008086d00006F05*
+ ID_MODEL_FROM_DATABASE=Broadwell PCI Express Root Port 2
+
+pci:v00008086d00006F06*
+ ID_MODEL_FROM_DATABASE=Broadwell PCI Express Root Port 2
+
+pci:v00008086d00006F07*
+ ID_MODEL_FROM_DATABASE=Broadwell PCI Express Root Port 2
+
+pci:v00008086d00006F08*
+ ID_MODEL_FROM_DATABASE=Broadwell PCI Express Root Port 3
+
+pci:v00008086d00006F09*
+ ID_MODEL_FROM_DATABASE=Broadwell PCI Express Root Port 3
+
+pci:v00008086d00006F0A*
+ ID_MODEL_FROM_DATABASE=Broadwell PCI Express Root Port 3
+
+pci:v00008086d00006F0B*
+ ID_MODEL_FROM_DATABASE=Broadwell PCI Express Root Port 3
+
+pci:v00008086d00006F10*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F11*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F12*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F13*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F14*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F15*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F16*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F17*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F18*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F19*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F1A*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F1B*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F1C*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Debug
+
+pci:v00008086d00006F1D*
+ ID_MODEL_FROM_DATABASE=Broadwell R2PCIe Agent
+
+pci:v00008086d00006F1E*
+ ID_MODEL_FROM_DATABASE=Broadwell Ubox
+
+pci:v00008086d00006F1F*
+ ID_MODEL_FROM_DATABASE=Broadwell Ubox
+
+pci:v00008086d00006F20*
+ ID_MODEL_FROM_DATABASE=Broadwell-DE Crystal Beach DMA Channel 0
+
+pci:v00008086d00006F21*
+ ID_MODEL_FROM_DATABASE=Broadwell-DE Crystal Beach DMA Channel 1
+
+pci:v00008086d00006F22*
+ ID_MODEL_FROM_DATABASE=Broadwell-DE Crystal Beach DMA Channel 2
+
+pci:v00008086d00006F23*
+ ID_MODEL_FROM_DATABASE=Broadwell-DE Crystal Beach DMA Channel 3
+
+pci:v00008086d00006F24*
+ ID_MODEL_FROM_DATABASE=Broadwell-DE Crystal Beach DMA Channel 4
+
+pci:v00008086d00006F25*
+ ID_MODEL_FROM_DATABASE=Broadwell-DE Crystal Beach DMA Channel 5
+
+pci:v00008086d00006F26*
+ ID_MODEL_FROM_DATABASE=Broadwell-DE Crystal Beach DMA Channel 6
+
+pci:v00008086d00006F27*
+ ID_MODEL_FROM_DATABASE=Broadwell-DE Crystal Beach DMA Channel 7
+
+pci:v00008086d00006F28*
+ ID_MODEL_FROM_DATABASE=Broadwell Adress Map/VTd_Misc/System Management
+
+pci:v00008086d00006F29*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO Hot Plug
+
+pci:v00008086d00006F2A*
+ ID_MODEL_FROM_DATABASE=Broadwell IIO RAS/Control Status/Global Errors
+
+pci:v00008086d00006F2C*
+ ID_MODEL_FROM_DATABASE=Broadwell I/O APIC
+
+pci:v00008086d00006F30*
+ ID_MODEL_FROM_DATABASE=Broadwell Home Agent 0
+
+pci:v00008086d00006F32*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 0
+
+pci:v00008086d00006F33*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 1
+
+pci:v00008086d00006F34*
+ ID_MODEL_FROM_DATABASE=Broadwell R2PCIe Agent
+
+pci:v00008086d00006F36*
+ ID_MODEL_FROM_DATABASE=Broadwell R3 QPI Link 0/1
+
+pci:v00008086d00006F37*
+ ID_MODEL_FROM_DATABASE=Broadwell R3 QPI Link 0/1
+
+pci:v00008086d00006F38*
+ ID_MODEL_FROM_DATABASE=Broadwell Home Agent 1
+
+pci:v00008086d00006F39*
+ ID_MODEL_FROM_DATABASE=Broadwell IO Performance Monitoring
+
+pci:v00008086d00006F3A*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 2
+
+pci:v00008086d00006F3E*
+ ID_MODEL_FROM_DATABASE=Broadwell R3 QPI Link 2
+
+pci:v00008086d00006F3F*
+ ID_MODEL_FROM_DATABASE=Broadwell R3 QPI Link 2
+
+pci:v00008086d00006F40*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 2
+
+pci:v00008086d00006F41*
+ ID_MODEL_FROM_DATABASE=Broadwell R3 QPI Link 2
+
+pci:v00008086d00006F43*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 2
+
+pci:v00008086d00006F45*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 2 Debug
+
+pci:v00008086d00006F46*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 2 Debug
+
+pci:v00008086d00006F47*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 2 Debug
+
+pci:v00008086d00006F60*
+ ID_MODEL_FROM_DATABASE=Broadwell Home Agent 1
+
+pci:v00008086d00006F68*
+ ID_MODEL_FROM_DATABASE=Broadwell Target Address/Thermal/RAS
+
+pci:v00008086d00006F6A*
+ ID_MODEL_FROM_DATABASE=Broadwell Channel Target Address Decoder
+
+pci:v00008086d00006F6B*
+ ID_MODEL_FROM_DATABASE=Broadwell Channel Target Address Decoder
+
+pci:v00008086d00006F6C*
+ ID_MODEL_FROM_DATABASE=Broadwell Channel Target Address Decoder
+
+pci:v00008086d00006F6D*
+ ID_MODEL_FROM_DATABASE=Broadwell Channel Target Address Decoder
+
+pci:v00008086d00006F6E*
+ ID_MODEL_FROM_DATABASE=Broadwell DDRIO Channel 2/3 Broadcast
+
+pci:v00008086d00006F6F*
+ ID_MODEL_FROM_DATABASE=Broadwell DDRIO Global Broadcast
+
+pci:v00008086d00006F70*
+ ID_MODEL_FROM_DATABASE=Broadwell Home Agent 0 Debug
+
+pci:v00008086d00006F71*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Target Address/Thermal/RAS
+
+pci:v00008086d00006F76*
+ ID_MODEL_FROM_DATABASE=Broadwell R3 QPI Link Debug
+
+pci:v00008086d00006F78*
+ ID_MODEL_FROM_DATABASE=Broadwell Home Agent 1 Debug
+
+pci:v00008086d00006F79*
+ ID_MODEL_FROM_DATABASE=Broadwell Target Address/Thermal/RAS
+
+pci:v00008086d00006F7D*
+ ID_MODEL_FROM_DATABASE=Broadwell Ubox
+
+pci:v00008086d00006F7E*
+ ID_MODEL_FROM_DATABASE=Broadwell R3 QPI Link Debug
+
+pci:v00008086d00006F80*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 0
+
+pci:v00008086d00006F81*
+ ID_MODEL_FROM_DATABASE=Broadwell R3 QPI Link 0/1
+
+pci:v00008086d00006F83*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 0
+
+pci:v00008086d00006F85*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 0 Debug
+
+pci:v00008086d00006F86*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 0 Debug
+
+pci:v00008086d00006F87*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 0 Debug
+
+pci:v00008086d00006F88*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006F8A*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006F90*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 1
+
+pci:v00008086d00006F93*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 1
+
+pci:v00008086d00006F95*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 1 Debug
+
+pci:v00008086d00006F96*
+ ID_MODEL_FROM_DATABASE=Broadwell QPI Link 1 Debug
+
+pci:v00008086d00006F98*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006F99*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006F9A*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006F9C*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FA0*
+ ID_MODEL_FROM_DATABASE=Broadwell Home Agent 0
+
+pci:v00008086d00006FA8*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Target Address/Thermal/RAS
+
+pci:v00008086d00006FAA*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Channel Target Address Decoder
+
+pci:v00008086d00006FAB*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Channel Target Address Decoder
+
+pci:v00008086d00006FAC*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Channel Target Address Decoder
+
+pci:v00008086d00006FAD*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Channel Target Address Decoder
+
+pci:v00008086d00006FAE*
+ ID_MODEL_FROM_DATABASE=Broadwell DDRIO Channel 0/1 Broadcast
+
+pci:v00008086d00006FAF*
+ ID_MODEL_FROM_DATABASE=Broadwell DDRIO Global Broadcast
+
+pci:v00008086d00006FB0*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Channel 0 Thermal Control
+
+pci:v00008086d00006FB1*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Channel 1 Thermal Control
+
+pci:v00008086d00006FB2*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Channel 0 Error
+
+pci:v00008086d00006FB3*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Channel 1 Error
+
+pci:v00008086d00006FB4*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Channel 2 Thermal Control
+
+pci:v00008086d00006FB5*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Channel 3 Thermal Control
+
+pci:v00008086d00006FB6*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Channel 2 Error
+
+pci:v00008086d00006FB7*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 0 - Channel 3 Error
+
+pci:v00008086d00006FB8*
+ ID_MODEL_FROM_DATABASE=Broadwell DDRIO Channel 2/3 Interface
+
+pci:v00008086d00006FB9*
+ ID_MODEL_FROM_DATABASE=Broadwell DDRIO Channel 2/3 Interface
+
+pci:v00008086d00006FBA*
+ ID_MODEL_FROM_DATABASE=Broadwell DDRIO Channel 2/3 Interface
+
+pci:v00008086d00006FBB*
+ ID_MODEL_FROM_DATABASE=Broadwell DDRIO Channel 2/3 Interface
+
+pci:v00008086d00006FBC*
+ ID_MODEL_FROM_DATABASE=Broadwell DDRIO Channel 0/1 Interface
+
+pci:v00008086d00006FBD*
+ ID_MODEL_FROM_DATABASE=Broadwell DDRIO Channel 0/1 Interface
+
+pci:v00008086d00006FBE*
+ ID_MODEL_FROM_DATABASE=Broadwell DDRIO Channel 0/1 Interface
+
+pci:v00008086d00006FBF*
+ ID_MODEL_FROM_DATABASE=Broadwell DDRIO Channel 0/1 Interface
+
+pci:v00008086d00006FC0*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FC1*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FC2*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FC3*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FC4*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FC5*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FC6*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FC7*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FC8*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FC9*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FCA*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FCB*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FCC*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FCD*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FCE*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FCF*
+ ID_MODEL_FROM_DATABASE=Broadwell Power Control Unit
+
+pci:v00008086d00006FD0*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 1 - Channel 0 Thermal Control
+
+pci:v00008086d00006FD1*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 1 - Channel 1 Thermal Control
+
+pci:v00008086d00006FD2*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 1 - Channel 0 Error
+
+pci:v00008086d00006FD3*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 1 - Channel 1 Error
+
+pci:v00008086d00006FD4*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 1 - Channel 2 Thermal Control
+
+pci:v00008086d00006FD5*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 1 - Channel 3 Thermal Control
+
+pci:v00008086d00006FD6*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 1 - Channel 2 Error
+
+pci:v00008086d00006FD7*
+ ID_MODEL_FROM_DATABASE=Broadwell Memory Controller 1 - Channel 3 Error
+
+pci:v00008086d00006FE0*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FE1*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FE2*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FE3*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FE4*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FE5*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FE6*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FE7*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FE8*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FE9*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FEA*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FEB*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FEC*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FED*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FEE*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FEF*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FF0*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FF1*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FF8*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FF9*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FFA*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FFB*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FFC*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FFD*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
+pci:v00008086d00006FFE*
+ ID_MODEL_FROM_DATABASE=Broadwell Caching Agent
+
 pci:v00008086d00007000*
  ID_MODEL_FROM_DATABASE=82371SB PIIX3 ISA [Natoma/Triton II]
 
@@ -73781,6 +74369,9 @@ pci:v0000D161d00001405*
 pci:v0000D161d00001420*
  ID_MODEL_FROM_DATABASE=Wildcard TE420 quad-span T1/E1/J1 card 3.3V (PCI-Express) (5th gen)
 
+pci:v0000D161d00001820*
+ ID_MODEL_FROM_DATABASE=Wildcard TE820 octal-span T1/E1/J1 card 3.3V (PCI-Express)
+
 pci:v0000D161d00002400*
  ID_MODEL_FROM_DATABASE=Wildcard TDM2400P 24-port analog card
 
@@ -73820,6 +74411,24 @@ pci:v0000D161d0000800A*
 pci:v0000D161d0000800B*
  ID_MODEL_FROM_DATABASE=Wildcard TE134 single-span T1/E1/J1 card
 
+pci:v0000D161d0000800C*
+ ID_MODEL_FROM_DATABASE=Wildcard A8A 8-port analog card
+
+pci:v0000D161d0000800D*
+ ID_MODEL_FROM_DATABASE=Wildcard A8B 8-port analog card (PCI-Express)
+
+pci:v0000D161d0000800E*
+ ID_MODEL_FROM_DATABASE=Wildcard TE235/TE435 quad-span T1/E1/J1 card (PCI-Express)
+
+pci:v0000D161d0000800F*
+ ID_MODEL_FROM_DATABASE=Wildcard A4A 4-port analog card
+
+pci:v0000D161d00008010*
+ ID_MODEL_FROM_DATABASE=Wildcard A4B 4-port analog card (PCI-Express)
+
+pci:v0000D161d00008013*
+ ID_MODEL_FROM_DATABASE=Wildcard TE236/TE436 quad-span T1/E1/J1 card
+
 pci:v0000D161d0000B410*
  ID_MODEL_FROM_DATABASE=Wildcard B410 quad-BRI card
 
diff --git a/hwdb/20-usb-vendor-model.hwdb b/hwdb/20-usb-vendor-model.hwdb
index 89cd5b1cfb..d87313cf87 100644
--- a/hwdb/20-usb-vendor-model.hwdb
+++ b/hwdb/20-usb-vendor-model.hwdb
@@ -524,6 +524,9 @@ usb:v03F0p0317*
 usb:v03F0p0324*
  ID_MODEL_FROM_DATABASE=SK-2885 keyboard
 
+usb:v03F0p034A*
+ ID_MODEL_FROM_DATABASE=Elite Keyboard
+
 usb:v03F0p0401*
  ID_MODEL_FROM_DATABASE=ScanJet 5200c
 
@@ -767,6 +770,9 @@ usb:v03F0p1317*
 usb:v03F0p1327*
  ID_MODEL_FROM_DATABASE=iLO Virtual Hub
 
+usb:v03F0p134A*
+ ID_MODEL_FROM_DATABASE=Optical Mouse
+
 usb:v03F0p1405*
  ID_MODEL_FROM_DATABASE=ScanJet 3670
 
@@ -2336,6 +2342,12 @@ usb:v0403pEF10*
 usb:v0403pF070*
  ID_MODEL_FROM_DATABASE=Serial Converter 422/485 [Vardaan VEUSB422R3]
 
+usb:v0403pF0C8*
+ ID_MODEL_FROM_DATABASE=SPROG Decoder Programmer
+
+usb:v0403pF0C9*
+ ID_MODEL_FROM_DATABASE=SPROG-DCC CAN-USB
+
 usb:v0403pF0E9*
  ID_MODEL_FROM_DATABASE=Tagsys L-P101
 
@@ -2642,6 +2654,9 @@ usb:v0409p02B4*
 usb:v0409p02B6*
  ID_MODEL_FROM_DATABASE=Aterm WL300NU-GS 802.11n Wireless Adapter
 
+usb:v0409p02BC*
+ ID_MODEL_FROM_DATABASE=Computer Monitor
+
 usb:v0409p0300*
  ID_MODEL_FROM_DATABASE=LifeTouch Note
 
@@ -5348,6 +5363,9 @@ usb:v0452p0050*
 usb:v0452p0051*
  ID_MODEL_FROM_DATABASE=Integrated Hub
 
+usb:v0452p0100*
+ ID_MODEL_FROM_DATABASE=Control Panel for Leica TCS SP5
+
 usb:v0453*
  ID_VENDOR_FROM_DATABASE=CMD Technology
 
@@ -8759,6 +8777,9 @@ usb:v0480pA009*
 usb:v0480pA00D*
  ID_MODEL_FROM_DATABASE=STOR.E BASICS 500GB
 
+usb:v0480pB001*
+ ID_MODEL_FROM_DATABASE=Stor.E Partner
+
 usb:v0480pD010*
  ID_MODEL_FROM_DATABASE=External Disk 3TB
 
@@ -11222,6 +11243,9 @@ usb:v04A9p3261*
 usb:v04A9p3262*
  ID_MODEL_FROM_DATABASE=PowerShot A2600
 
+usb:v04A9p3263*
+ ID_MODEL_FROM_DATABASE=PowerShot SX275 HS
+
 usb:v04A9p3264*
  ID_MODEL_FROM_DATABASE=PowerShot A1400
 
@@ -11240,6 +11264,9 @@ usb:v04A9p3276*
 usb:v04A9p3277*
  ID_MODEL_FROM_DATABASE=PowerShot SX510 HS
 
+usb:v04A9p3278*
+ ID_MODEL_FROM_DATABASE=PowerShot S200
+
 usb:v04A9p327D*
  ID_MODEL_FROM_DATABASE=Powershot ELPH 115 IS / IXUS 132
 
@@ -11645,6 +11672,9 @@ usb:v04B4p2050*
 usb:v04B4p2830*
  ID_MODEL_FROM_DATABASE=Opera1 DVB-S (cold state)
 
+usb:v04B4p3813*
+ ID_MODEL_FROM_DATABASE=NANO BIOS Programmer
+
 usb:v04B4p4235*
  ID_MODEL_FROM_DATABASE=Monitor 02 Driver
 
@@ -13100,9 +13130,15 @@ usb:v04D8p900A*
 usb:v04D8pC001*
  ID_MODEL_FROM_DATABASE=PicoLCD 20x4
 
+usb:v04D8pF4B5*
+ ID_MODEL_FROM_DATABASE=SmartScope
+
 usb:v04D8pF8DA*
  ID_MODEL_FROM_DATABASE=Hughski Ltd. ColorHug
 
+usb:v04D8pF91C*
+ ID_MODEL_FROM_DATABASE=SPROG IIv3
+
 usb:v04D8pFAFF*
  ID_MODEL_FROM_DATABASE=Dangerous Prototypes BusPirate v4 Bootloader mode
 
@@ -13118,6 +13154,9 @@ usb:v04D8pFBBA*
 usb:v04D8pFBBB*
  ID_MODEL_FROM_DATABASE=DiscFerret Magnetic Disc Analyser (active mode)
 
+usb:v04D8pFC1E*
+ ID_MODEL_FROM_DATABASE=Bachrus Speedometer Interface
+
 usb:v04D8pFC92*
  ID_MODEL_FROM_DATABASE=Open Bench Logic Sniffer
 
@@ -14432,6 +14471,9 @@ usb:v04F2p0403*
 usb:v04F2p0418*
  ID_MODEL_FROM_DATABASE=KU-0418 Tactical Pad
 
+usb:v04F2p0618*
+ ID_MODEL_FROM_DATABASE=RG-0618U Wireless HID Receiver & KG-0609 Wireless Keyboard with Touchpad
+
 usb:v04F2p0760*
  ID_MODEL_FROM_DATABASE=Acer KU-0760 Keyboard
 
@@ -15336,7 +15378,7 @@ usb:v04FCp0171*
  ID_MODEL_FROM_DATABASE=SPCA1527A/SPCA1528 SD card camera (Mass Storage mode)
 
 usb:v04FCp0201*
- ID_MODEL_FROM_DATABASE=RS232C Adapter
+ ID_MODEL_FROM_DATABASE=SPCP825 RS232C Adapter
 
 usb:v04FCp0232*
  ID_MODEL_FROM_DATABASE=Fingerprint
@@ -29036,6 +29078,9 @@ usb:v08E3p0301*
 usb:v08E4*
  ID_VENDOR_FROM_DATABASE=Pioneer Corp.
 
+usb:v08E4p0185*
+ ID_MODEL_FROM_DATABASE=DDJ-WeGO2
+
 usb:v08E5*
  ID_VENDOR_FROM_DATABASE=Litronic
 
@@ -33080,6 +33125,12 @@ usb:v0B48p300D*
 usb:v0B48p300E*
  ID_MODEL_FROM_DATABASE=TT-connect C-2400
 
+usb:v0B48p3012*
+ ID_MODEL_FROM_DATABASE=TT-connect CT2-4650 CI
+
+usb:v0B48p3014*
+ ID_MODEL_FROM_DATABASE=TT-TVStick CT2-4400
+
 usb:v0B49*
  ID_VENDOR_FROM_DATABASE=ASCII Corp.
 
@@ -34086,7 +34137,7 @@ usb:v0BC2p3101*
  ID_MODEL_FROM_DATABASE=FreeAgent XTreme 640GB
 
 usb:v0BC2p3312*
- ID_MODEL_FROM_DATABASE=SRD00F2 [Expansion Desktop Drive 2TB]
+ ID_MODEL_FROM_DATABASE=SRD00F2 Expansion Desktop Drive (STBV)
 
 usb:v0BC2p3320*
  ID_MODEL_FROM_DATABASE=SRD00F2 [Expansion Desktop Drive]
@@ -34109,6 +34160,9 @@ usb:v0BC2p5031*
 usb:v0BC2p5070*
  ID_MODEL_FROM_DATABASE=FreeAgent GoFlex Desk
 
+usb:v0BC2p5071*
+ ID_MODEL_FROM_DATABASE=FreeAgent GoFlex Desk
+
 usb:v0BC2p50A1*
  ID_MODEL_FROM_DATABASE=FreeAgent GoFlex Desk
 
@@ -34133,6 +34187,9 @@ usb:v0BC2pA0A4*
 usb:v0BC2pAB00*
  ID_MODEL_FROM_DATABASE=Slim Portable Drive
 
+usb:v0BC2pAB20*
+ ID_MODEL_FROM_DATABASE=Backup Plus Portable Drive
+
 usb:v0BC3*
  ID_VENDOR_FROM_DATABASE=IPWireless, Inc.
 
@@ -34301,6 +34358,9 @@ usb:v0BDAp0176*
 usb:v0BDAp0178*
  ID_MODEL_FROM_DATABASE=Mass Storage Device
 
+usb:v0BDAp0179*
+ ID_MODEL_FROM_DATABASE=RTL8188ETV Wireless LAN 802.11n Network Adapter
+
 usb:v0BDAp0184*
  ID_MODEL_FROM_DATABASE=RTS5182 Card Reader
 
@@ -34322,6 +34382,9 @@ usb:v0BDAp2832*
 usb:v0BDAp2838*
  ID_MODEL_FROM_DATABASE=RTL2838 DVB-T
 
+usb:v0BDAp5730*
+ ID_MODEL_FROM_DATABASE=HP 2.0MP High Definition Webcam
+
 usb:v0BDAp8150*
  ID_MODEL_FROM_DATABASE=RTL8150 Fast Ethernet Adapter
 
@@ -36012,11 +36075,53 @@ usb:v0CE5p0003*
  ID_MODEL_FROM_DATABASE=Matrix
 
 usb:v0CE9*
- ID_VENDOR_FROM_DATABASE=pico Technology
+ ID_VENDOR_FROM_DATABASE=Pico Technology
 
 usb:v0CE9p1001*
  ID_MODEL_FROM_DATABASE=PicoScope3000 series PC Oscilloscope
 
+usb:v0CE9p1007*
+ ID_MODEL_FROM_DATABASE=PicoScope 2000 series PC Oscilloscope
+
+usb:v0CE9p1008*
+ ID_MODEL_FROM_DATABASE=PicoScope 5000 series PC Oscilloscope
+
+usb:v0CE9p1009*
+ ID_MODEL_FROM_DATABASE=PicoScope 4000 series PC Oscilloscope
+
+usb:v0CE9p100E*
+ ID_MODEL_FROM_DATABASE=PicoScope 6000 series PC Oscilloscope
+
+usb:v0CE9p1012*
+ ID_MODEL_FROM_DATABASE=PicoScope 3000A series PC Oscilloscope
+
+usb:v0CE9p1016*
+ ID_MODEL_FROM_DATABASE=PicoScope 2000A series PC Oscilloscope
+
+usb:v0CE9p1018*
+ ID_MODEL_FROM_DATABASE=PicoScope 4000A series PC Oscilloscope
+
+usb:v0CE9p1200*
+ ID_MODEL_FROM_DATABASE=PicoScope 2000 series PC Oscilloscope
+
+usb:v0CE9p1201*
+ ID_MODEL_FROM_DATABASE=PicoScope 3000 series PC Oscilloscope
+
+usb:v0CE9p1202*
+ ID_MODEL_FROM_DATABASE=PicoScope 4000 series PC Oscilloscope
+
+usb:v0CE9p1203*
+ ID_MODEL_FROM_DATABASE=PicoScope 5000 series PC Oscilloscope
+
+usb:v0CE9p1204*
+ ID_MODEL_FROM_DATABASE=PicoScope 6000 series PC Oscilloscope
+
+usb:v0CE9p1211*
+ ID_MODEL_FROM_DATABASE=PicoScope 3000 series PC Oscilloscope
+
+usb:v0CE9p1212*
+ ID_MODEL_FROM_DATABASE=PicoScope 4000 series PC Oscilloscope
+
 usb:v0CF1*
  ID_VENDOR_FROM_DATABASE=e-Conn Electronic Co., Ltd
 
@@ -36998,6 +37103,9 @@ usb:v0DB0p1020*
 usb:v0DB0p1967*
  ID_MODEL_FROM_DATABASE=Bluetooth Dongle
 
+usb:v0DB0p3713*
+ ID_MODEL_FROM_DATABASE=Primo 73
+
 usb:v0DB0p3801*
  ID_MODEL_FROM_DATABASE=Motorola Bluetooth 2.1+EDR Device
 
@@ -37112,6 +37220,9 @@ usb:v0DB7p0002*
 usb:v0DBA*
  ID_VENDOR_FROM_DATABASE=Digidesign
 
+usb:v0DBAp1000*
+ ID_MODEL_FROM_DATABASE=Mbox 1 [Mbox]
+
 usb:v0DBAp3000*
  ID_MODEL_FROM_DATABASE=Mbox 2
 
@@ -37913,6 +38024,12 @@ usb:v0E6A*
 usb:v0E6Ap0101*
  ID_MODEL_FROM_DATABASE=MA100 [USB-UART Bridge IC]
 
+usb:v0E6Ap030B*
+ ID_MODEL_FROM_DATABASE=Truly Ergonomic Computer Keyboard (Device Firmware Update mode)
+
+usb:v0E6Ap030C*
+ ID_MODEL_FROM_DATABASE=Truly Ergonomic Computer Keyboard
+
 usb:v0E6Ap6001*
  ID_MODEL_FROM_DATABASE=GEMBIRD Flexible keyboard KB-109F-B-DE
 
@@ -39548,6 +39665,12 @@ usb:v1039p0824*
 usb:v1039p2140*
  ID_MODEL_FROM_DATABASE=dsl+ 1100 duo
 
+usb:v103A*
+ ID_VENDOR_FROM_DATABASE=PSA
+
+usb:v103ApF000*
+ ID_MODEL_FROM_DATABASE=Actia Evo XS
+
 usb:v103D*
  ID_VENDOR_FROM_DATABASE=Stanton
 
@@ -39692,6 +39815,9 @@ usb:v1050p0110*
 usb:v1050p0111*
  ID_MODEL_FROM_DATABASE=Yubikey NEO OTP+CCID
 
+usb:v1050p0200*
+ ID_MODEL_FROM_DATABASE=U2F Gnubby
+
 usb:v1050p0211*
  ID_MODEL_FROM_DATABASE=Gnubby
 
@@ -39735,16 +39861,25 @@ usb:v1058p0704*
  ID_MODEL_FROM_DATABASE=Passport External HDD
 
 usb:v1058p070A*
- ID_MODEL_FROM_DATABASE=My Passport Essential SE, My Passport for Mac (WDBAAB)
+ ID_MODEL_FROM_DATABASE=My Passport Essential (WDBAAA), My Passport for Mac (WDBAAB), My Passport Essential SE (WDBABM), My Passport SE for Mac (WDBABW)
+
+usb:v1058p070B*
+ ID_MODEL_FROM_DATABASE=My Passport Elite (WDBAAC)
+
+usb:v1058p070C*
+ ID_MODEL_FROM_DATABASE=My Passport Studio (WDBAAE)
 
 usb:v1058p071A*
- ID_MODEL_FROM_DATABASE=My Passport
+ ID_MODEL_FROM_DATABASE=My Passport Essential (WDBAAA)
+
+usb:v1058p071D*
+ ID_MODEL_FROM_DATABASE=My Passport Studio (WDBALG)
 
 usb:v1058p0730*
  ID_MODEL_FROM_DATABASE=My Passport Essential (WDBACY)
 
 usb:v1058p0740*
- ID_MODEL_FROM_DATABASE=My Passport
+ ID_MODEL_FROM_DATABASE=My Passport Essential (WDBACY)
 
 usb:v1058p0741*
  ID_MODEL_FROM_DATABASE=My Passport Ultra
@@ -39756,22 +39891,22 @@ usb:v1058p0748*
  ID_MODEL_FROM_DATABASE=My Passport (WDBKXH)
 
 usb:v1058p0810*
- ID_MODEL_FROM_DATABASE=My Passport Ultra
+ ID_MODEL_FROM_DATABASE=My Passport Ultra (WDBZFP)
 
 usb:v1058p0900*
  ID_MODEL_FROM_DATABASE=MyBook Essential External HDD
 
 usb:v1058p0901*
- ID_MODEL_FROM_DATABASE=My Book Essential Edition (Green Ring)
+ ID_MODEL_FROM_DATABASE=My Book Essential Edition (Green Ring) (WDG1U)
 
 usb:v1058p0902*
- ID_MODEL_FROM_DATABASE=My Book Pro Edition
+ ID_MODEL_FROM_DATABASE=My Book Pro Edition (WDG1T)
 
 usb:v1058p0903*
  ID_MODEL_FROM_DATABASE=My Book Premium Edition
 
 usb:v1058p0910*
- ID_MODEL_FROM_DATABASE=MyBook Essential External HDD
+ ID_MODEL_FROM_DATABASE=My Book Essential Edition (Green Ring) (WDG1U)
 
 usb:v1058p1001*
  ID_MODEL_FROM_DATABASE=Elements Desktop (WDE1U)
@@ -39786,7 +39921,7 @@ usb:v1058p1021*
  ID_MODEL_FROM_DATABASE=Elements Desktop (WDBAAU)
 
 usb:v1058p1023*
- ID_MODEL_FROM_DATABASE=Elements SE
+ ID_MODEL_FROM_DATABASE=Elements SE Portable (WDBABV)
 
 usb:v1058p1048*
  ID_MODEL_FROM_DATABASE=Elements Portable (WDBU6Y)
@@ -39809,11 +39944,23 @@ usb:v1058p1104*
 usb:v1058p1105*
  ID_MODEL_FROM_DATABASE=My Book Studio II
 
+usb:v1058p1110*
+ ID_MODEL_FROM_DATABASE=My Book Essential (WDBAAF), My Book for Mac (WDBAAG)
+
+usb:v1058p1111*
+ ID_MODEL_FROM_DATABASE=My Book Elite (WDBAAH)
+
+usb:v1058p1112*
+ ID_MODEL_FROM_DATABASE=My Book Studio (WDBAAJ), My Book Studio LX (WDBACH)
+
 usb:v1058p1123*
- ID_MODEL_FROM_DATABASE=My Book 3.0
+ ID_MODEL_FROM_DATABASE=My Book 3.0 (WDBABP)
+
+usb:v1058p1130*
+ ID_MODEL_FROM_DATABASE=My Book Essential (WDBACW)
 
 usb:v1058p1140*
- ID_MODEL_FROM_DATABASE=My Book Essential USB3.0
+ ID_MODEL_FROM_DATABASE=My Book Essential (WDBACW)
 
 usb:v1059*
  ID_VENDOR_FROM_DATABASE=Giesecke & Devrient GmbH
@@ -42098,6 +42245,12 @@ usb:v132Bp2045*
 usb:v132Bp2049*
  ID_MODEL_FROM_DATABASE=Magicolor 2490MF
 
+usb:v133E*
+ ID_VENDOR_FROM_DATABASE=Kemper Digital GmbH
+
+usb:v133Ep0815*
+ ID_MODEL_FROM_DATABASE=Virus TI Desktop
+
 usb:v1342*
  ID_VENDOR_FROM_DATABASE=Mobility
 
@@ -42203,6 +42356,9 @@ usb:v134E*
 usb:v1357*
  ID_VENDOR_FROM_DATABASE=P&E Microcomputer Systems
 
+usb:v1357p0089*
+ ID_MODEL_FROM_DATABASE=OpenSDA - CDC Serial Port
+
 usb:v1357p0503*
  ID_MODEL_FROM_DATABASE=USB-ML-12 HCS08/HCS12 Multilink
 
@@ -42213,7 +42369,7 @@ usb:v1366*
  ID_VENDOR_FROM_DATABASE=SEGGER
 
 usb:v1366p0101*
- ID_MODEL_FROM_DATABASE=J-Link ARM
+ ID_MODEL_FROM_DATABASE=J-Link PLUS
 
 usb:v136B*
  ID_VENDOR_FROM_DATABASE=STEC
@@ -42311,6 +42467,9 @@ usb:v138Ap003C*
 usb:v138Ap003D*
  ID_MODEL_FROM_DATABASE=VFS491
 
+usb:v138Ap003F*
+ ID_MODEL_FROM_DATABASE=VFS495 Fingerprint Reader
+
 usb:v138Ap0050*
  ID_MODEL_FROM_DATABASE=Swipe Fingerprint Sensor
 
@@ -42647,6 +42806,9 @@ usb:v13D3p5122*
 usb:v13D3p5126*
  ID_MODEL_FROM_DATABASE=PC Cam
 
+usb:v13D3p5130*
+ ID_MODEL_FROM_DATABASE=Integrated Webcam
+
 usb:v13D3p5702*
  ID_MODEL_FROM_DATABASE=UVC VGA Webcam
 
@@ -43232,6 +43394,9 @@ usb:v1493p0019*
 usb:v1493p001A*
  ID_MODEL_FROM_DATABASE=Colibri [Ambit2 S]
 
+usb:v1493p001D*
+ ID_MODEL_FROM_DATABASE=Greentit [Ambit2 R]
+
 usb:v1497*
  ID_VENDOR_FROM_DATABASE=Panstrong Company Ltd.
 
@@ -49640,12 +49805,39 @@ usb:v2478*
 usb:v2478p2008*
  ID_MODEL_FROM_DATABASE=U209-000-R Serial Port
 
+usb:v249C*
+ ID_VENDOR_FROM_DATABASE=M2Tech s.r.l.
+
 usb:v2632*
  ID_VENDOR_FROM_DATABASE=TwinMOS
 
 usb:v2632p3209*
  ID_MODEL_FROM_DATABASE=7-in-1 Card Reader
 
+usb:v2639*
+ ID_VENDOR_FROM_DATABASE=Xsens
+
+usb:v2639p0001*
+ ID_MODEL_FROM_DATABASE=MTi-10 IMU
+
+usb:v2639p0002*
+ ID_MODEL_FROM_DATABASE=MTi-20 VRU
+
+usb:v2639p0003*
+ ID_MODEL_FROM_DATABASE=MTi-30 AHRS
+
+usb:v2639p0011*
+ ID_MODEL_FROM_DATABASE=MTi-100 IMU
+
+usb:v2639p0012*
+ ID_MODEL_FROM_DATABASE=MTi-200 VRU
+
+usb:v2639p0013*
+ ID_MODEL_FROM_DATABASE=MTi-300 AHRS
+
+usb:v2639p0017*
+ ID_MODEL_FROM_DATABASE=MTi-G-700 GPS INS
+
 usb:v2650*
  ID_VENDOR_FROM_DATABASE=Electronics For Imaging, Inc. [hex]
 
